---
title: "Functional Analyses of Distinct Oxygen Concentrations and Light Niches of *Prochlorococcus marinus*"
author:
  - name: '**Mireille Savoie**'
    affil: 1
    main: true
    email: msavoie@mta.ca
  - name: Julie Nadeau
    affil: 1
  - name: Laurel Genge
    affil: 1
  - name: Sylwia Śliwińska-Wilczewska
    affil: 1,2
  - name: Maximilian Berthold
    affil: 1
  - name: Naaman Omar
    affil: 1
  - name: Douglas A. Campbell
    affil: 1
    main: false
    orcid: ''
    email: 'dcampbel@mta.ca'
affiliation:
  - num: 1
    address: Department of Biology, Mount Allison University
institute:
  - mta: 'Department of Biology, Mount Allison University, 53 York St., Sackville NB, Canada, E4L 1C9.'
date: "`r format(Sys.Date())`"
output:
  bookdown::html_document2:
    code_folding: show
    keep_md: yes
    toc: TRUE
    toc_float: TRUE
    toc_depth: 6
    fig_caption: yes
    pandoc_args: 
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
  bookdown::word_document2:
    code_folding: show
    reference_docx: KnitTemplate.docx
    keep_md: yes
    toc: TRUE
    toc_depth: 6
    fig_caption: yes
    pandoc_args: 
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
always_allow_html: true
bibliography: [Manuscript_O2.bib, RPackages.bib]
csl: plos-one.csl
editor_options: 
  markdown: 
    wrap: 72
---

```{css, echo=FALSE}
p.caption {
  font-size: 12px;
}
```

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE, message = FALSE, warning = FALSE)
knitr::opts_chunk$set(fig.path='Figs/')
```

```{r load libraries, include = FALSE}
# libraries; Note check actual dependencies

library(lubridate)
library(broom)
library(knitr)
library(grid)
library(bookdown)
library(glue)
library(cowplot)
library(ggpubr)
# library(RColorBrewer)
library(tidyverse)
library(png)
library(cowplot)

knitr::write_bib(c("tidyverse", "lubridate","knitr",  "broom", "zoo", "photobiologyWavebands", "RColorBrewer", "data.table","googlesheets4",  "grid", "glue", "bookdown", "photobiology", "mgcv", "minpack.lm", "ggh4x", "cowplot", "egg"), 'RPackages.bib')

```

```{r set project variables, include = FALSE}
#"..", takes up a level in the directory path
Project <- "PICO"
DataIn <- file.path("..", "Data", "CleanData")
FiguresIn <- file.path("..","Output", "Figures")
FiguresInOceanPro <- file.path( "..", "..", "OceanProteinProchloro", "Output")
FigureInO2Analy <- file.path(  "..", "..", "O2Analyses", "Output")

```

```{r read ProcessFile, include = FALSE}

# PURPARFits <- readRDS(file.path(DataIn, "PICO_Merged_GrowthFitsPURPAR.Rds"))
 
```

```{r, echo=FALSE, results='asis'}
citations <- list()
knitr::knit_hooks$set(knitcitations = function(before, options, envir) {
  if (before) {
    citations$reflist <- options$reflist
    citations$section <- options$section
  }
})
```

# Abstract {.unnumbered}


\newpage

# Introduction {.unnumbered}




\newpage

# Materials and methods {.unnumbered}

## MetaProteomics {.unnumbered}

The OceansMap Protein Portal (OPP; https://proteinportal.whoi.edu/) is an open access online data repository (Woods Hole Oceanographic Institute, WHOI) of mass spectroscopy data on marine microbial peptides, sampled from various depths and locations worldwide. We screened a subset of the OPP for proteins annotated as from *Prochlorococcus* strains, to identify differential strategies employed by strains living at varying depths and oxygen levels within the marine water column.  We focussed on proteins mediating photosynthesis and protein metabolism. The samples for metaproteomic analyses were collected from 12 locations in the tropical North Pacific ocean along 150 W from 18 N of the equator between October 1, 2011 and October 25, 2011 during the voyage of the R/V Kilo Moana MetZyme cruise KM1128 (https://www.rvdata.us/search/cruise/KM1128; original datasets in the Biological and Chemical Oceanography Data Management Office repository; https://www.bco-dmo.org/project/2236). 

Collection and treatment of protein samples were performed following Saito et al., (2014, 2015) XXXXADD to ZoteroXXXX . Briefly, samples of seawater from depths of 20, 40, 50, 60, 70, 80, 90, 120, 150, 200, 250, 300, 380, 400, 500, 550, 600, and 800 m below the ocean surface were pumped through a 0.2 µm filter, preserved in RNAlater and frozen at -80°C until extraction. Proteins were extracted from the filter in an SDS-based detergent, embedded in tube gel, alkylated and reduced prior to in-gel trypsin digestion. Peptide spectra were generated using a Q-Exactive Orbitrap Mass Spectrometer, searched in the SEQUEST CITATIONXXXX and labelled with the most likely protein and species annotation from Uniprot.  Oxygen levels at the location of sampling were recorded. 

## *Prochlorococcus* culturing and experimental design {.unnumbered}

Three xenic cultures of *P. marinus* were obtained from Bigelow Labs, NCMA
Maine, US (REFERENCE). MED4 (CCMP1986) is from High Light-adapted (HLI) clade; SS120
(CCMP1375) is from Low Light-adapted (LLIII) clade; and MIT9313 (CCMP2773)
is from Low Light-adapted (LLIV) clade. Cultures were
maintained in incubators set to 22°C with a light/dark cycle of 12 h, with illumination from XXXXX. The PAR level for maintenance cultures reflected  Photosynthetically Active Radiation (PAR) in the source
niche of the ecotype; MED4, of 160 µmol photons m^-2^ s^-1^; SS120 and
MIT9313 at 30 µmol photons m^-2^ s^-1^. To maintain active growth all strains were
transferred weekly with XX to YY dilution with Pro99 media
[@mooreCulturingMarineCyanobacterium2007] prepared with autoclaved artificial
seawater (National
Center for Marine Algae and Microbiota (NCMA) protocol by combining salt
solution I and salt solution II using the enriched artificial seawater
(ESAW) recipe.  REPLACe with CITATION OF NCMA protocol)XXXX

Controlled growth experiments were performed using PSI Multicultivators (Figure \@ref(fig:LabeledMC); MCMIX-OD orMC1000-OD, Brno, Czech Republic ).  Each multicultivator individually controls 8 tubes at a common temperature of 22°C.  Each tube containing 70 mL of Pro99 media was inoculated with 10 mL of growing maintenance culture. In a factorial matrix design, each tube was then subject to an individual combination of sinusoidal photoperiod (4, 8, 12, 16  h);  reaching a peak PAR (30, 90, 180 µmol photons m^-2^ s^-1^), with defined spectral bandwidth (AA, BB, CC, DD nm).  [O~2~] levels (2.5 µM, 25 µM, 250 µM) were imposed by bubbling tubes with varying ratios of air and
Nitrogen (N~2~), with consistent 0.05% of Carbon Dioxide (CO~2~)
gas, delivered through a 0.2 μm sterile microfilter via a G400 gas mixing system XXXXGIVE SOURCEXXXX. [O~2~] in situ was verified using oxygen optodes (PyroScience,
Germany) inserted into tubes for
real-time measurements, with a temperature probe in
the aquarium of the bioreactor to correct [O~2~] measures for temperature
fluctuations. In addition, the Pyroscience software corrected [O~2~] based on the
salinity of the media (32 ppt). The flow rate of the gas mixture was controlled, but variations in bubbling speed, PAR and culture density affected the [O~2~] achieved in each tube. A low [O~2~] of 0.5 µM - 5 µM (reported as 2.5 µM hereafter),  was achieved by sparging with a gas mixture containing 99.95% N~2~ and 0.05%
CO~2~. An intermediate [O~2~] of 10 - 25 µM (reported hereafter as 25 µM) was achieved by sparging with a gas mixture containing 98.95% N~2~, 0.05% CO~2~ and 1% O~2~. A high O~2~ of 200 µM - 280 µM (reported hereafter as 250 µM) was achieved by sparging with lab air (78% N~2~, 21% O~2~, 1% Ar and 0.05% CO~2~).

The full crossing of all factor levels would yield 4 x 3 x 4 x 3 = 144 treatments, x 3 strains for 432 possible combinations. Consistent absence of growth of some strains under some levels of photoperiod, PAR, or [O~2~] meant we completed 291 growth factor treatment combinations. 

In situ XXXXItalicsXXX measurements of Optical Density (OD) 680 nm, a proxy for cell suspension density, cell size dependent scatter and cell chlorophyll content;  and OD 720 nm, a proxy for cell suspension density and cell size dependent scatter, were recorded every 5 minutes over least 8 to 14 days,  depending on the
duration of the lag phase, if any. All data from the Multicultivator were saved
as comma separated values file  (CITATION of FILES LOCATION).

PAR of 180, 90 or 30 µmol photons m^-2^ s^-1^, and spectral wavebands (white LED
full spectrum, 660 nm, 530 nm, and 450 nm) were chosen to approximate light
levels and spectral colors spanning the vertical ocean water column, from
near-surface to the lower euphotic zone depths. Photoperiods were chosen to approximate diel cycles characteristic of current and hypothetical future
niches of *P. marinus*; 16 h represents
temperate (45°N) summer at the ocean surface; 12 h for equatorial (0°N)
ocean surface or temperate (45°N) spring and fall ocean surface or
temperate (45°N) summer at deeper ocean depths; 8 h for temperate (45°N)
winter at the surface or at temperate (45°N) spring and fall at depth
and equatorial (0°N) deep ocean depths; and 4 h for temperate (45°N)
winter or deep ocean depths during temperate (45°N) spring and fall.


```{r LabeledMC, fig.cap = LabeledMC_cap , fig.height = 8, fig.width = 6, echo = FALSE}

LabeledMC_cap <- glue("**PSI MCMIX-OD Multicultivator.**",
                      "Spectral wavebands and light levels are individually controlled for each culture tube.",
                      "Real time Optical Density (OD) measurements eliminate intrusive subsampling of sterile cultures.",
                      "The temperature of culture tubes are controlled via heating or cooling of the aquarium water.",
                      "Gas with specific oxygen concentrations are bubbled through a humidifier and passed through a 0.2 um filter.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "LabeledMC.png"))
```

\newpage

## MetaProteomics bioinformatic analyses and limitations  {.unnumbered}

	Metaproteomic datasets were obtained from the KM1128 entry in the BCO-DMO database (https://www.bco-dmo.org/deployment/59053) accessed via the OPP in June 2019. Datasets contained:
	i) Peptide sequences and sample identification (ID) number;
	ii) Sample ID number, station, depth in meters below the surface the sample was collected at, best-hit BLASTP protein and species annotation and the corresponding Uniprot Entry number for the identified proteins;
	iii) Sample station depth and [O~2].  
	The depth and [O~2] were joined to peptide sequence and BLASTP annotations by ID number, depth and station using the functions in the Tidyverse package (https://www.tidyverse.org/) in RStudio v1.2.5019 (https://rstudio.com/)  (LOCATION OF CODE ON GITHUBO. The resulting merged dataset was filtered for *Prochlorococcus* peptides, detected from 0 to 300 m below the surface, annotated as a subunit of *Prochlorococcus* chlorophyll binding proteins (Pcb); Photosystem II (PSII); Cytochrome b6f (Cytb6f);  Photosystem I (PSI); NADPH Dehydrogenase (NDH); Terminal Oxidase (PTOX); plastocyanin (PC); ferredoxin (Fd); Ribulose-1,5-bisphosphate oxygenase (RUBISCO);  ATP Synthase;  FtsH proteases (FtsH) or ribosomes. 
	Detected peptides were re-annotated for consistency and labelled according to strain, clade, subunit and protein complex (where possible). Full protein sequences corresponding to detected proteins were obtained from UniProt (https://www.uniprot.org/) and analyzed in Molecular Evolution and Genetic Analyses X (MEGAX) software  (https://www.megasoftware.net/). Sequences for each protein and/or subunits for each of the thirteen Prochlorococcus strains identified in the dataset were aligned with MUSCLE using UPGMA cluster method and a lambda of 24 with a -2.9 gap open penalty and 1.20 hydrophobicity multiplier.  Overall mean pairwise distance between protein sequences was determined using bootstrap variance estimation methods. Maximum likelihood phylogenetic trees were assembled using 1000 bootstrap replications with a 95% site coverage cut off. *Prochlorococcus* FtsH isoform identities were specified by phylogenetic comparison to the characterized FtsH proteases of Synechocystis sp. PCC6803 (CITATONS).  Data for each strain was plotted against depth and  [O~2] and sampling station.
	
	Graphs depict the detection of mass-spectrally detected peptides at varying depths and [O~2] in the water column for  *Prochlorococcus* strains and clades identified in the dataset. When assessing the presence of a particular protein complex, all peptides belonging to all subunits of the complex were included to give the greatest number of data points. 
	Although not a direct measurement, it is assumed that light intensity is inversely correlated with depth. 
	As this data was acquired on a discovery mission rather than through targeted peptide approaches, it is difficult to discern accuracies of strain annotations, particularly as the proteins of interest in this study are highly conserved in sequence across strain. We are confident in clade and ecotypic classification for each protein examined, although exact strain annotation may be ambiguous.
Another caveat to interpretation of this data is the “peptide detection bias” inherent to mass spectrometry: certain peptides are more easily detected and thus dominate the spectra over less easily detected peptides. The data is also limited by the number and nature of protein spectra in the SEQUEST database: a peptide sequence cannot be determined unless there is already a known spectrum for that peptide in the SEQUEST database, hence some peptides of interest may not be identifiable. Therefore, the absence of a particular does not necessarily rule out its presence overall.  Furthermore, a peptide must be detected above a certain threshold abundance in order to be considered an accurate ‘hit’. Therefore, the ‘absence’ of a particular protein structure is here interpreted as low in quantity, while its presence is interpreted as higher in quantity.

## Growth data management and analysis {.unnumbered}
Data files saved form from the Multicultivator software were imported into R-Studio (CITATION) for data management [@R-tidyverse], growth rate calculations CITATION, comparisons of model fits CITATION, and generation of
figures`ggplot2` CITATION. The chlorophyll proxy
optical density (OD680 - OD720;  ΔOD) was used to determine the chlorophyll-specific exponential growth
rate (µ, h^-1^) for each treatment combination.  We first we used a rolling mean from the RStudio `zoo` package [@R-zoo] to calculate the average ΔOD data over a 1-hour window, to lower influence of outlier points XXXXother trimmingXXXXX using a Levenberg-Marquardt algorithm
[@bellaviaLevenbergMarquardtMethod2018] modification of the non-linear
least squares,  using the R package `minpack.lm`
[@R-minpack.lm], to fit a logistic
equation \@ref(eq:GrowthRate) where ΔOD~max~ is maximum ΔOD, ΔOD~min~ is minimum ΔOD, t is time
duration over the growth trajectory. 
OTHER MOREXXXXXX

```{=tex}
\begin{equation}
  µ = \frac{ΔOD_{max} × ΔOD_{min} × exp^{(µ × t)}}{ΔOD_{max} + (ΔOD_{min} × exp^{((µ × t) - 1)})}
  (\#eq:GrowthRate)
\end{equation}
```

CITAtIONS of github locations of code etc. XXXXX

### Generalized additive model of growth responses {.unnumbered}

%%%%%% Taken from my thesis word for word, need to reword and shorten %%%%

A Generalized Additive Model (GAM) was applied to examine the
relationship between chlorophyll-specific µ, h^-1^, for each [O~2~] level, across the blue and red spectral wavebands, photoperiods and PAR levels for each *P.marinus* strain in this study. The gam function from the R package `mgcv`
[@R-mgcv] was used to model the growth rate with smoothing terms to
indicate the 90, 50 and 10% quantiles for growth rate across the levels of factors. Only data  XXXXXWHAT DATAXXXX BE SPECIFICXXXX below a standard error tolerance of 30% of the fit was used in the model. Our priority
was studying the effect of blue light on growth trends, since blue light is the most ecologically relevant spectral waveband for
deep ocean niches.  We included analyses of responses of red light, which is not ecophysiologically relevant but might prove mechanistically informative (CITE MURPHY ETC)

### Estimation of Photosynthetically Usable Radiation

XXXXTEXT NEEDEDXXXX
```{r PurParPlot, fig.cap = PurParPlot_cap , fig.height = 20, fig.width = 10, echo = FALSE}

PurParPlot_cap <- glue("**The absorbed Photosynthetically Usable Radiation (PUR) (µmol photons m^-2^ s^-1^) vs. the Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^).**",
                       "The correlation between PAR, plotted on the x-axis and absorbed PUR, plotted on the y-axis, are colored for each growth spectral waveband; blue for 450 nm, red for 660 nm and black for white LED full spectrum light.",
                       "The grey dashed line represents a hypothetical one to one correlation.",
                       "**A.** is *Prochlorococcus marinus*  MED4.",
                       "**B.** is *Prochlorococcus marinus*  SS120.",
                       "**C.** is *Prochlorococcus marinus*  MIT9313.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PurParPlot.png"))
```


\newpage

# Results and Discussion {.unnumbered}

%%%% OceanProteinPortal data show complexes in environment  %%%


```{r ProchloroPhotosynthDepthO2, fig.cap = ProchloroPhotosynthDepthO2_cap , fig.height = 20, fig.width = 10, echo = FALSE}

ProchloroPhotosynthDepthO2_cap <- glue("**Ocean detection of *Prochlorococcus marinus* photosynthesis complexes.**", 
                       "Protein detections are plotted vs. O~2~ (µM) (X axis) and depth (m) (Y axis) at sample origin.",
                       "Rows separate data annotated as from *Prochlorococcus marinus* strains MED4, SS120 and MIT9313.",
                       "Columns show detections of proteins annotated as  Photosystem II (PSII), Cytochromeb6f complex (Cytb6f), Photosystem I (PSI), ATP Synthase or Ribulose-1,5-bisphosphate oxygenase carboxylase (RUBISCO).",
                       "Culture growth experimental conditions indicated by horizontal grey lines for depths approximating Photosynthetically Active Radiation (µmol photons m^-2^ s^-1^) and vertical grey lines for O~2~ (µM).",
                       "Data from OceanProteinPortal (https://www.oceanproteinportal.org/).",
 .sep = " ")

knitr::include_graphics(file.path(FiguresInOceanPro, "ProchloroPhotosynthDepthO2.png"))
```

\newpage

%%%% Insert GAM figures to show lab results confirming Ocean Protein Portal environment results %%%


```{r MED4GAM,  fig.cap = MED4GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

MED4GAM_cap <- glue("**A contour plot of a Generalized Additive Model (GAM) representing the chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* MED4 grown under 660 nm (red) and 450 nm (blue) light.**",
                    "The x-axis is photoperiod (h).",
                    "The y-axis is actinic Photosynethetic Active Radiation (PAR) level (µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 250 µM of O~2~ and blue light.",
                    "**D.** represents the model under 25 µM of O~2~ and blue light.",
                    "The legend represents a colour gradient of growth rate from no growth (white) to 1 d^-1^ (dark blue).",
                    "The labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_MED4_Oxy_.png"))


```

\newpage


```{r SS120GAM,  fig.cap = SS120GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

SS120GAM_cap <- glue("**A contour plot of a Generalized Additive Model (GAM) representing the chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* SS120 grown under 660 nm (red) and 450 nm (blue) light.**",
                    "The x-axis is photoperiod (h).",
                    "The y-axis is actinic Photosynethetic Active Radiation (PAR) level (µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 250 µM of O~2~ and blue light.",
                    "**D.** represents the model under 25 µM of O~2~ and blue light.",
                    "The legend represents a colour gradient of growth rate from no growth (white) to 1 d^-1^ (dark blue).",
                    "The labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_SS120_Oxy_.png"))


```

\newpage

```{r MIT9313GAM,  fig.cap = MIT9313GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

MIT9313GAM_cap <- glue("**A contour plot of a Generalized Additive Model (GAM) representing the chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* MIT9313 grown under 660 nm (red) and 450 nm (blue) light.**",
                    "The x-axis is photoperiod (h).",
                    "The y-axis is actinic Photosynthetically Active Radiation (PAR) level (µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 2.5 µM of O~2~ and red light.",
                    "**D.** represents the model under 250 µM of O~2~ and blue light.",
                    "**E.** represents the model under 25 µM of O~2~ and blue light.",
                    "**F.** represents the model under 2.5 µM of O~2~ and blue light.",
                    "The legend represents a colour gradient of growth rate from no growth (white) to 1 d^-1^ (dark blue).",
                    "The labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_MIT9313_Oxy_.png"))


```

\newpage

%%%% Insert PURFits figures  %%%


\newpage

%%% TO DO Fits and ANOVA %%
```{r MED4PAR, fig.cap = MED4PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MED4PAR_cap <- glue("**Chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* MED4 (High Light (HLI) near surface clade) vs. photoperiod (h).  **",  
                    
                    "3 levels of growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) are in columns;  3 levels of imposed growth dissolved O~2~ concentrations (µM) are in rows.",
                    "Colors represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MED4PAR.png"))
```

\newpage

%%% TO DO Fits and ANOVA %%
```{r SS120PAR, fig.cap = SS120PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

SS120PAR_cap <- glue("**Chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* SS120 (Low Light (LLIII) deep ocean clade) vs. photoperiod (h). **",  
                     
                    "3 levels of growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) are in columns;  3 levels of imposed growth dissolved O~2~ concentrations (µM) are in rows.",
                    "Colors represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",
                    
                       .sep = " ")
                       
knitr::include_graphics(file.path(FiguresIn, "SS120PAR.png"))

```

\newpage

%%% TO DO Fits and ANOVA %%
```{r MIT9313PAR, fig.cap = MIT9313PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MIT9313PAR_cap <- glue("**Chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* MIT9313 (Low Light (LLIV) deep ocean clade) vs. photoperiod (h). **",  
                       
                    "3 levels of growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) are in columns;  3 levels of imposed growth dissolved O~2~ concentrations (µM) are in rows.",
                    "Colors represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",

                       .sep = " ")
                       
knitr::include_graphics(file.path(FiguresIn, "MIT9313PAR.png"))

```

\newpage

%%%% OceanFtsHRibosomeProteins figure from OceanPortal repository  %%%


```{r ProchloroProteinMetabDepthO2, fig.cap = ProchloroProteinMetabDepthO2_cap , fig.height = 20, fig.width = 10, echo = FALSE}

ProchloroProteinMetabDepthO2_cap <- glue("**Ocean detection of *Prochlorococcus marinus* protein turnover complexes.**", 
                       "Protein detections are plotted vs. O~2~ (µM) (X axis) and depth (m) (Y axis) at sample origin.",
                       "Rows separate data annotated as from *Prochlorococcus marinus* strains MED4, SS120 and MIT9313.",
                       "Columns show detections of proteins annotated as FtsH Protease Complexes (FtsH1, FtsH2, FtsH3) or the Ribosome).",
                       "Culture growth experimental conditions indicated by horizontal grey lines for depths approximating Photosynthetically Active Radiation (µmol photons m^-2^ s^-1^) and vertical grey lines for O~2~ (µM).",
                       "Data from OceanProteinPortal (https://www.oceanproteinportal.org/).",
 .sep = " ")



knitr::include_graphics(file.path(FiguresInOceanPro, "ProchloroMetabDepthO2.png"))
```






```{r MED4ProteinAnotated, fig.cap = MED4ProteinAnotated_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MED4ProteinAnotated_cap <- glue("**fmole target protein per ug total protein for *Prochlorococcus marinus* MED4 (High Light (HLI) near surface clade).  **",  
                    
                    "Growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) and spectral wavelength are in rows;  2 levels of imposed growth dissolved O~2~ concentrations (µM) are in columns.",
                    "Numbers over each bar are fmole/ug",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MED4ProteinAnotated.png"))
```

\newpage



```{r MIT9313ProteinAnotated, fig.cap = MIT9313ProteinAnotated_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MIT9313ProteinAnotated_cap <- glue("**fmole target protein per ug total protein for *Prochlorococcus marinus* MIT9313 (Low Light (LLIV) deep ocean clade).  **",  
                    
                    "Growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) and spectral wavelength are in rows;  2 levels of imposed growth dissolved O~2~ concentrations (µM) are in columns.",
                    "Numbers over each bar are fmole/ug",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MIT9313ProteinAnotated.png"))
```

\newpage


# Supplemental {.unnumbered}



# References {.unnumbered}