---
title: "*Prochlorococcus marinus* responses to light and oxygen" 
author:
  - name: '**Mireille Savoie**'
    affil: 1
    main: true
    email: msavoie@mta.ca
  - name: Aurora Mattison
    affil: 1,2
  - name: Julie Nadeau
    affil: 1
  - name: Laurel Genge
    affil: 1
  - name: Sylwia Śliwińska-Wilczewska
    affil: 1
  - name: Maximilian Berthold
    affil: 1,3
  - name: Naaman Omar
    affil: 1
  - name: Ondřej Prášil
    affil: 1,4
    main: false
  - name: Amanda M. Cockshutt
    affil: 1,5
    main: false
  - name: Douglas A. Campbell
    affil: 1
    main: false
    orcid: '0000-0001-8996-5463'
    email: 'dcampbel@mta.ca'
affiliation:
  - num: 1
    address: Department of Biology, Mount Allison University
institute:
  - mta: 'Department of Biology, Mount Allison University, 53 York St., Sackville NB, Canada, E4L 1C9.'
  - num: 2
institute:
  - ubc: 'Department of Biology, University of British Columbia'
  - num: 3
institute:
  - umwelt: 'Umweltbundesamtm, Dessau-Roßlau, Germany'
  - num: 4
institute:
  - alga: 'Algatech, Czech Academy of Science'
  - num: 5
institute:
  - sfx: 'Department of Chemistry, St. Frances Xavier University'
date: "`r format(Sys.Date())`"
output:
  bookdown::html_document2:
    code_folding: show
    keep_md: yes
    toc: TRUE
    toc_float: TRUE
    toc_depth: 6
    fig_caption: yes
    pandoc_args: 
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
  bookdown::word_document2:
    code_folding: show
    reference_docx: KnitTemplate.docx
    keep_md: yes
    toc: TRUE
    toc_depth: 6
    fig_caption: yes
    pandoc_args: 
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
always_allow_html: true
bibliography: [Manuscript_O2.bib, RPackages.bib]
csl: plos-one.csl
editor_options: 
  markdown: 
    wrap: 72
---

```{css, echo=FALSE}
p.caption {
  font-size: 12px;
}
```

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE, message = FALSE, warning = FALSE)
knitr::opts_chunk$set(fig.path='Figs/')
```

```{r load libraries, include = FALSE}
# libraries; Note check actual dependencies

library(lubridate)
library(broom)
library(knitr)
library(grid)
library(bookdown)
library(glue)
library(cowplot)
library(ggpubr)
# library(RColorBrewer)
library(tidyverse)
library(png)
library(cowplot)

knitr::write_bib(c("tidyverse", "lubridate","knitr",  "broom", "zoo", "photobiologyWavebands", "RColorBrewer", "data.table","googlesheets4",  "grid", "glue", "bookdown", "photobiology", "mgcv", "minpack.lm", "ggh4x", "cowplot", "egg"), 'RPackages.bib')

```

```{r set project variables, include = FALSE}
#"..", takes up a level in the directory path
Project <- "PICO"
DataIn <- file.path("..", "Data", "CleanData")
FiguresIn <- file.path("..","Output", "Figures")
FiguresInOceanPro <- file.path( "..", "..", "OceanProteinProchloro", "Output")
FigureInO2Analy <- file.path(  "..", "..", "O2Analyses", "Output")

```

```{r read ProcessFile, include = FALSE}

# PURPARFits <- readRDS(file.path(DataIn, "PICO_Merged_GrowthFitsPURPAR.Rds"))
 
```

```{r, echo=FALSE, results='asis'}
citations <- list()
knitr::knit_hooks$set(knitcitations = function(before, options, envir) {
  if (before) {
    citations$reflist <- options$reflist
    citations$section <- options$section
  }
})
```

# Abstract {.unnumbered}


\newpage

# Introduction {.unnumbered}

## Mireille
### Chapter 1 General Intro

*Prochlorococcus marinus*, a genus of the phylum Cyanobacteria, is the smallest known photosynthetic prokaryote, with cell diameters ranging from 0.5 to 0.7 µm [@chisholmProchlorococcusMarinusNov1992]. Despite a small cell size, *P. marinus* contribute 13 to 48% of net primary production in oligotrophic oceans, corresponding to about 30% of  global oxygen production [@partenskyProchlorococcusMarinePhotosynthetic1999].  

*Prochlorococcus marinus* comprises many strains, organized into clades,
which in turn are grouped into ecotypes defined largely as High-Light (HL) or Low-Light (LL) adapted,  and aligned phylogenetically [@moorePhysiologyMolecularPhylogeny1998]. The biomass distribution of *P. marinus* is constrained to between latitudes of 40°N to 40°S in open ocean waters [@partenskyProchlorococcusMarinePhotosynthetic1999] with different ecotypes inhabiting distinct ecological niches. High-Light ecotypes are typically observed as the dominant picophytoplankton in near-surface oligotrophic waters that are, characterized by low nutrient concentrations, and high light levels. The monophyletic high light ecotype is subdivided into clades based on iron adaptation, whereby HLIII/IV are low iron adapted, whereas HLI is a high iron, low temperature adapted clade and HLII is a high iron, high temperature adapted clade [@kentParallelPhylogeographyProchlorococcus2019; @johnsonNichePartitioningProchlorococcus2006; @zinserInfluenceLightTemperature2007].  Low-Light ecotypes thrive in deeper ocean waters, extending beyond 200 m in depth [@partenskyProchlorococcusMarinePhotosynthetic1999]. In these deeper regions, only 1% of the surface irradiance penetrates, primarily in the blue (450 nm) to green (520 nm) spectral waveband ranges, due to light and spectral attenuation [@holtropVibrationalModesWater2021]. Additionally, some *P. marinus* strains in clade LLV and LLVI thrive in  Oxygen Minimum Zones (OMZ) of the subtropical Atlantic and Pacific Oceans, where dissolved oxygen concentrations [O~2~] can be less than 20 µM [@ulloaCyanobacteriumProchlorococcusHas2021]. Currently there are at least 12 clades but only 5 have been cultivated; HLI, HLII, LLI, LLII/III and LIV [@billerProchlorococcusStructureFunction2015].




Our changing climate, characterized by warming ocean waters, is rapidly altering conditions for these highly specialized strains of marine picophytoplankton. Predictions indicate a net global increase of *P. marinus* cell abundances of 29% and poleward latitudinal shifts of at least 10° in marine phytoplankton niches by the end of this century [@flombaumPresentFutureGlobal2013] in response to warming waters, with cell abundance increases of approximately 50% in the more poleward regions. As the solar incidence angles are lower and photoperiod seasonality is more pronounced at higher latitudes, this shift will carry these phytoplankters into new combinations of photoperiod, seasonal regimes and changing spectral regimes [@bartonAnthropogenicClimateChange2016]. A 10° poleward shift of *P. marinus*, coupled with a dramatic increase in cell abundance between 40°N and 50°N,  could have a large impact on ocean ecosystems and global biogeochemical cycles [@flombaumPresentFutureGlobal2013], and thus will undoubtedly impact the ecosystem balance off our Canadian waters.

Global warming is also rapidly changing the chemistry of our oceans.  By the end of this century, surface ocean pH is projected to decline by 0.1 to 0.4  due to  projected increases in carbon dioxide concentrations [@garcia-sotoOverviewOceanClimate2021]. Moreover, substantial changes in the global water cycle, leading to extensive changes in worldwide precipitation patterns, are affecting ocean salinity levels on a global scale, and  ice melts due to rising temperatures are impacting salinity levels in the Arctic and Northwest Atlantic oceans [@leeRecognizingSalinityThreats2022]. There is evidence that increasing sea temperatures are also causing a decrease in [O~2~] across our global oceans [@matearLongtermChangesDissolved2003], with significant declines observed toward the poles in both hemispheres [@helmObservedDecreasesOxygen2011]. Warmer ocean waters decrease oxygen solubility at the surface which in turn reduces the amount of oxygen that can be mixed downwards by ocean currents [@garcia-sotoOverviewOceanClimate2021]. Furthermore, models predict that the OMZ in the Pacific and Indian Oceans are expanding [@garcia-sotoOverviewOceanClimate2021; @buseckeDivergingFatesPacific2022]. In light of these expanding OMZ, a recent study by @buseckeDivergingFatesPacific2022 highlights an intriguing finding; the core of the OMZ, where the oxygen levels are lowest, is projected to contract.  Given the predicted decrease in overall oceanic [O~2~],   projected expansion of OMZs, and possibly decline in the OMZ core volumes, we set forth to examine the influence of [O~2~] on *P. marinus* growth as this area of research has been relatively understudied.



To determine the viability of *P. marinus* strains in changing ecosystems, we analyzed the growth and physiological responses of representative strains from three clades to varying [O~2~], light levels, spectral waveband ranges, and photoperiods,  to simulate a matrix of eco-physiological conditions representative of current and hypothetical future ocean zones. The following *P. marinus* clades were examined: *P. marinus* MED4, classified as a clade HLI (Figure \@ref(fig:PhylogenyTreeYan2018edit)), was isolated near the ocean surface (5m depth) of the Mediterranean Sea where [O~2~] are near saturation, light levels are high and spectral bias from full solar irradiance is minimal. *P. marinus* SS120, a clade LLIII (Figure \@ref(fig:PhylogenyTreeYan2018edit)), was isolated from the Sargasso Sea at a depth of 120 m, while *P. marinus* MIT9313, a clade LLIV (Figure \@ref(fig:PhylogenyTreeYan2018edit)), was isolated from the North Atlantic Gulf Stream at a depth of 135 m. At these depths, light attenuation and spectral shifts occur, resulting in low blue and green light levels, while [O~2~] varies from near-surface saturation levels to decreased concentrations, but it does not necessarily decrease with depth [@billheimerOxygenSeasonalityUtilization2021]. In chapter 2, we define the interactions between oxygen, light level, light colour and photoperiod that help define their complex niches.  

In chapter 3, to better decipher the physiological processes driving the observed growth responses to a complex matrix of spectral light regimes and [O~2~], we express growth rates in terms of cumulative diel Photosynthetically Usable Radiation (PUR). Photosynthetic organisms rely on absorbing light energy within the visible spectrum, 350 to 700 nm, for photosynthesis. This range of light, known as Photosynthetic Active Radiation (PAR) [@morelAvailableUsableStored1978], is essential for the photosynthetic process. PUR represents the fraction of PAR that is absorbed by the pigments of photosynthetic organisms [@morelAvailableUsableStored1978], thus we can compare light levels across different spectral wavebands that are absorbed differently by cells. Moreover, expressing growth rates in terms of cumulative diel absorbed PUR has the potential to simplify different photoperiods, spectral bands, and PAR levels into a common parameter, making growth response comparisons across different oxygen levels more attainable and may allow us to determine whether growth responses are primarily influenced by cumulative diel absorbed PUR, specific photoperiods or spectral bands.

This study focuses on evaluating how growth of *Prochlorococcus* strains from different clades responds to varying [O~2~], spectral wavebands, light levels and photoperiods. 


### Chapter 2 PAR Intro
*Prochlorococcus marinus* ecotypes are characterized by their distinct
tolerance for High-Light (HL) or Low-Light (LL) levels
[@moorePhysiologyMolecularPhylogeny1998]. *Prochlorococcus marinus* is
classified into at least 12 major clades
[@billerProchlorococcusStructureFunction2015], that respectively inhabit
a range of ecological niches across vertical ocean water columns
[@chisholmNovelFreelivingProchlorophyte1988;
@veldhuisVerticalDistributionPigment1990;
@partenskyProchlorococcusMarinePhotosynthetic1999].

Previous studies have observed that the diversification of *P. marinus*
niches extends into regions considered unfavorable for photosynthesis to
occur. @goerickeNovelNicheProchlorococcus2000 first reported the
existence of *P. marinus* below the oxyclines of the Oxygen Minimum
Zones (OMZ) of the Arabian Sea and the Eastern Tropical North Pacific.
In an OMZ, oxygen concentrations [O~2~] can be less than 20 µM, and in
some regions [O~2~] is undetectable
[@garcia-robledoCrypticOxygenCycling2017a]. Subsequent research has
revealed that *P. marinus* LL ecotypes dominate the OMZ
[@lavinNovelLineagesProchlorococcus2010;
@ulloaCyanobacteriumProchlorococcusHas2021] and are believed to
contribute to approximately 90% of the *P. marinus* diversity in the
eastern tropical Pacific Ocean OMZ
[@partenskyProchlorococcusAdvantagesLimits2010].

Although *P. marinus* ecotypes are defined by their ecological niches,
there is evidence to suggest that ecotypes can exist in environments
beyond their optimal habitats. HL ecotypes are known to
inhabit depths where LL ecotypes also inhabit
[@westNichePartitioningProchlorococcusPopulationsStratified1999;
@zinserInfluenceLightTemperature2007;
@delmontLinkingPangenomesMetagenomes2018] but at light levels still
within the range of confirmed growth irradiance limits
[@moorePhotophysiologyMarineCyanobacterium1999]. Studies by
@lavinNovelLineagesProchlorococcus2010 and
@mattisonPhotosyntheticStrategiesWild2020 show evidence that LL ecotypes
can occupy regions in the OMZ at depths above 40 m. Since light levels
were not reported at every depth of these studies, it is presumed that
at a depth of 40 m, closer to the surface, light levels are higher than
what LL ecotypes can tolerate. Furthermore,
@mattisonPhotosyntheticStrategiesWild2020 presented evidence of proteins
derived from HL ecotypes of *P. marinus* in the OMZ at depths up to 200
meters with O~2~ of 15 µM. However, the extent to which [O~2~] defines
the niches occupied by different *P. marinus* ecotypes, as compared to
potentially covarying environmental variables like photoperiod, light
spectrum, and light level, is poorly described.

Photosynthetic organisms rely on the absorption of light energy within
the visible spectral range of 350 to 700 nm to drive the photosynthetic
process. This spectral range of light, known as Photosynthetic Active
Radiation (PAR) [@morelAvailableUsableStored1978], is essential for
photosynthesis. However, as sunlight enters the marine environment, it
undergoes attenuation, resulting in a decline of specific wavebands of
light through the water column from the surface. This attenuation
primarily affects the longer wavelengths in the red region of the
spectrum (600 to 700 nm), followed by wavelengths in the green region
(500 to 600 nm) and, finally, wavelengths in the blue region (400 to 500
nm) [@holtropVibrationalModesWater2021]. Thus, as sunlight penetrates the
marine environment, the light level diminishes and the residual spectral
wavebands shift with depth, where only blue spectral wavelengths
dominate. The region of the water column with enough light to support
photosynthesis is known as the photic zone, defined as the depth where
light levels remain above 20 µmol photons m^-2^ s^-1^, 1% of the surface
solar incident radiation, of 2000 µmol photons m^-2^ s^-1^.

In growth studies of *P. marinus*, it is common to use a broad spectrum
light source to approximate the solar spectral wavelengths found in the
uppermost layers of the ocean. The actinic photon flux is adjusted to
simulate the specific light levels appropriate for the ecotype of
interest. However, this approach falls short when attempting to
accurately reproduce the ambient spectral light quality of deep ocean
habitats, where only blue spectral wavelengths prevail
[@holtropVibrationalModesWater2021]. *Prochlorococcus marinus* contain a
unique light-harvesting complex composed primarily of divinyl
chlorophyll a and b [@chisholmProchlorococcusMarinusNov1992;
@goerickePigmentsProchlorococcusMarinus1992], the only photosynthetic
organism to possess these chlorophyll derivatives. The absorption maxima
of 442 nm for divinyl chlorophyll a and 478 nm for divinyl chlorophyll b
[@goerickePigmentsProchlorococcusMarinus1992] allow *P. marinus* to
efficiently harvest the light energy emitted by blue light in the 400 nm
to 500 nm range [@morelAvailableUsableStored1978].

At most latitudes, the duration of the photoperiod (day length)
undergoes seasonal variations. However, at the equator, the photoperiod
remains relatively constant at the ocean surface, approximately 12 hours
(h) of daylight and 12 h of darkness throughout the entire year. As *P.
marinus* potentially expands its temperature-permissive niches poleward
into temperate regions due to climate change
[@flombaumPresentFutureGlobal2013;
@bartonAnthropogenicClimateChange2016], it will encounter more
pronounced seasonal variations in photoperiod regimes. Photoperiods are
longer at temperate regions during the summer, while in spring, fall and
winter, photoperiods are shorter. Additionally, when considering the
influence of light attenuation with depth, temperate regions exhibit
more combinations of photoperiod compared to equatorial regions. The
effects of photoperiod on the ecophysiology of phytoplankton is
underscored by several studies. @liDiatomGrowthResponses2017
demonstrated complex interactions of photoperiod and growth rate of
diatoms. Similarly, @prezelinDielPeriodicityPhytoplankton1992 expressed
the importance of considering photoperiod on *in situ* rates of primary
production, as the diel light to dark ratio regulates metabolic
processes throughout the day. Furthermore,
@vaulotGrowthProchlorococcusPhotosynthetic1995 showed that
*Prochlorococcus* undergoes DNA replication in the afternoon, while cell
division occurs at night. Further research is needed to investigate the
impact of different photoperiods on the growth of *Prochlorococcus*, as
this aspect remains inadequately studied.

\newpage

(fig:SunDepth) shows light levels and spectral waveband
attenuation at ocean surface to ocean depths of 1000 m, as well as the
hours of light above the minimum photic zone of 20 µmol photons m^-2^
s^-1^, representing the photoperiod. This figure is modeled using the
solar incident angle at 45°N, a hypothetical future niche, at summer
solstice and illustrates the light parameters that define the
photoregimes of the niches of *P. marinus* that we studied. Figure (fig:CardinalSeasons) illustrates light attenuation through ocean waters at 45°N at winter solstice, equinox and summer solstice.

### Chapter 3 Intro (PUR)

*Prochlorococcus marinus* ecotypes are characterized by their distinct
tolerance for High-Light (HL) or Low-Light (LL) levels. The photosynthetic strategies between *Prochlorococcus marinus* ecotypes must differ in order to exploit the very diverse photo regimes in their respective niches and may be explained by their unique cellular compositions. 
*Prochlorococcus marinus* MED4 has the smallest genome size of the 3 ecotypes we studied, only 1.66 Mbp, while *P. marinus* MIT9313 has the largest genome size of 2.41 Mbp [@rocapGenomeDivergenceTwo2003]. *Prochlorococcus marinus* SS120 genome size is 1.75 Mbp [@dufresneGenomeSequenceCyanobacterium2003]. Due to the small genome size and small cell diameters, ranging from 0.5 to 0.7 µm [@chisholmProchlorococcusMarinusNov1992], *Prochlorococcus* are the smallest known photosynthetic prokaryote. The simplistic cellular structure and small size of *P. marinus* minimizes the pigment package effect or intracellular self-shading, due to the high surface area to volume ratio, contributing to very efficient optical absorption capabilities [@morelProchlorococcusSynechococcusComparative1993].


Photosynthetic organisms depend on the absorption of light energy in the visible spectrum, 350 to 700 nm, to carry out photosynthesis. This range of light, referred to as Photosynthetic Active Radiation (PAR) [@morelAvailableUsableStored1978], is crucial for the photosynthetic process. The Photosynthetic Usable Radiation (PUR) is defined as the fraction of PAR absorbed by the pigments of a given photosynthetic taxon [@morelAvailableUsableStored1978]. PUR, therefore, takes into account the pigment composition of photosynthetic organisms and the specific spectral wavebands that these pigments absorb. *Prochlorococcus marinus* MED4, a clade HLI ecotype, dominate the near ocean surface where spectral attenuation from full solar irradiance is minimal and light levels are high [@partenskyProchlorococcusMarinePhotosynthetic1999]. *Prochlorococcus marinus* SS120, a clade LLIII ecotype, and *P. marinus* MIT9313, a clade LLIV ecotype, dominate the deep ocean [@partenskyProchlorococcusMarinePhotosynthetic1999] where extreme light attenuation and spectral shifts occur, resulting in the availability of very low light levels in the blue (450 nm) to green (520 nm) spectral range [@holtropVibrationalModesWater2021]. Given the different spectral light regimes of the different ecotype niches, understanding how the quantity of available light levels, PAR, converts to what these ecotypes can use, PUR, can give some insight into how they are able to exploit light energy.


The photosynthesis process can be divided into two distinct reactions: the
light-dependent and the light-independent reactions. *Prochlorococcus marinus* does not contain
chloroplasts [@chisholmNovelFreelivingProchlorophyte1988], and the
light-dependent reactions occur in thylakoid membranes arranged concentrically around the periphery of the cell. Light harvesting complexes spanning a thylakoid membrane capture or 'harvest' light
energy. The light harvesting complexes in *P. marinus* consist of Prochlorophyte Chlorophyll Binding
proteins (Pcb) [@larocheIndependentEvolutionProchlorophyte1996]. Once photons are absorbed by light harvesting pigments, the energy is transferred to the reaction center of Photosystem II
(PSII) where water is split, catalyzed by the manganese cluster of the
oxygen evolving complex. Here, 2 water molecules are oxidized to produce 1 oxygen
(O~2~) molecule, releasing 4 protons into the thylakoid lumen, which contribute to the transmembrane proton
potential,  and 4 electrons. These electrons are passed to Photosystem I
(PSI) through a series of reduction and oxidation reactions in the
electron transport chain in the thylakoid membrane, and in the process, deliver more protons to the
proton pool within the lumen side of the thylakoid membrane. Photosystem I
(PSI) also harvest light energy but does not split water, rather the
reaction center uses excitation energy to excite electrons received from PSII
and transfer them by the reduction and oxidation of iron-sulphur
clusters to reduce NADP^+^ to NADPH. The Adenosine Triphosphate (ATP) synthase complex produces ATP in the stroma, driven by protons passing from the lumen out to the stroma side of the thylakoid
membrane. The light-independent reaction involves the RuBisCO enzyme
which uses the NADPH and ATP generated in the light-dependent reaction
to convert dissolved carbon dioxide (CO~2~) gas into carbohydrates. 


*Prochlorococcus marinus* is differentiated from other cyanobacteria as a distinct lineage, primarily due to the absence of phycobilisomes [@chisholmProchlorococcusMarinusNov1992]. Instead, *Prochlorococcus*  utilizes intra-membrane Pcb proteins bound to divinylchlorophyll a (Chl a2), and α-carotene, as the major light harvesting complex [@goerickePigmentsProchlorococcusMarinus1992]; [@larocheIndependentEvolutionProchlorophyte1996].  Not all *Prochlorococcus* encode the same number of Pcb proteins. The representative of HLI clade, *P. marinus* MED4 has 1 gene encoding pcbA protein [@garczarekMultiplicationAntennaGenes2000] and the LLIV clade, *P. marinus* MIT9313, has 2 genes encoding pcbB and pcbG proteins [@rocapGenomeDivergenceTwo2003]. The LLIII clade, *P. marinus* SS120, however, has 7 genes encoding pcbA, pcbB, pcbC, pcbE, pcbF, pcbG and pcbH [@garczarekMultiplicationAntennaGenes2000]. According to @garczarekMultiplicationAntennaGenes2000, the expression of some of the pcb genes might be specifically induced under low light conditions, after noting the thickening of the antenna band of the thylakoid membrane when SS120 was switched to low light after high light exposure. Another difference within the cell structure among *P. marinus* ecotypes is the 2 to 4 thylakoids that run parallel to the cell membrane. For SS120, the thylakoids are closed around the periphery of the cell; however, in MED4 and MIT9313, there is a gap resulting in a horseshoe shape for the thylakoids [@partenskyProchlorococcusMarinePhotosynthetic1999].  Despite the deviation in the light harvesting complex from cyanobacteria, the genes encoding the rest of photosynthetic apparatus is highly conserved in *Prochlorococcus* [@kettlerPatternsImplicationsGene2007], though the photosynthetic efficiency may vary between ecotypes [@hessPhotosyntheticApparatusProchlorococcus2001; @moorePhotophysiologyMarineCyanobacterium1999].


The efficient ability of *P. marinus* to harvest light energy allows LL ecotypes to thrive and exploit the low, attenuated light levels present in deep ocean waters. However, this efficiency also makes them vulnerable to photo damage under high light levels [@aroPhotoinhibitionPhotosystemII1993; @soitamoPhotoinhibitionMarinePicocyanobacteria2017]. The proteins in the reaction centers of PSII are vulnerable to photo damage, which can result in the disruption of electron transport and irreversible damage to the reaction center subunit D1 protein [@murphyPhotoinactivationPhotosystemII2017]. Additionally, direct absorption of a photon in the UV or blue range by the Mn~4~Ca cluster within the oxygen-evolving complex of PSII can lead to its inactivation [@hakalaEvidenceRoleOxygenevolving2005]. Without the proper functioning of the oxygen-evolving complex, the P680^+^ radical, which is formed during the light absorption process, cannot be efficiently reduced. This can result in the accumulation of P680^+^ radicals and lead to irreversible oxidative damage to the PsbA protein, within PSII. The damaged PsbA protein must then be removed to reactivate the PSII center. This process relies on the degradation and removal of damaged PsbA by protease isoforms FtsH 2 and 3 and the production of newly synthesized PsbA [@nixonRecentAdvancesUnderstanding2010]. Generation of Reactive Oxygen Species (ROS) can also cause photoinhibition if the electron flow is slowed downstream of the P680^+^ radical [@aroPhotoinhibitionPhotosystemII1993], producing  singlet oxygen (^1^O~2~).


There are metabolic processes occurring in both the light-dependent and light-independent reactions of photosynthesis that are greatly influenced by the duration of light exposure [@liDiatomGrowthResponses2017]; [@prezelinDielPeriodicityPhytoplankton1992]. @vaulotGrowthProchlorococcusPhotosynthetic1995 showed that
*Prochlorococcus* replication of DNA occurs in the afternoon and cell division occurs at night. It is often overlooked that the duration of dark exposure is just as important as light exposure for photosynthetic organisms.


Converting the quantity of ambient light levels, PAR, to absorbed PUR, makes it possible to directly compare light levels across different spectral wavebands that are differentially absorbed by the cells.  This conversion enables us to assess and compare the amount of light that can be effectively utilized for photosynthesis by specific ecotypes in different spectral regimes. Furthermore, expressing cumulative diel absorbed PUR can potentially collapse different photoperiods, spectral bands and PAR levels to a common denominator, allowing us to detect whether growth responses are driven simply by cumulative diel absorbed PUR, or whether specific photoperiods, spectral bands or PAR levels have independent effects on growth. We can draw meaningful insights by evaluating the growth responses based on cumulative diel absorbed PUR across different oxygen levels that represent distinct ecological niches. 

## Aurora
*Prochlorococcus marinus*, a species of picocyanobacteria, are tiny, photosynthetic, marine prokaryotes. At nder 1 µm in diameter, they are the smallest photosynthetic organisms on the planet, but at 10^4^-10^5^ cells per millilitre of seawater  (ampbell and Vaulot, 1993), they heavily influence global biogeochemical cycles (Partensky et al., 1999) (Flombaum et al., 2013). *Prochlorococcus* are currently restricted to between 40°N and 40°S, from surface to 300 m  depth, spanning 3 orders of magnitudes of light (Partensky et al., 1999) (Chisholm et al., 1992). Some *Prochlorococcus* extend into Oxygen Minimum Zones (OMZ), at less than  20 µM O2 (Johnson et al., 1999). *Prochlorococcus* is divided into numerous strains, categorized into high-light clades HLI, HLII, HLIII or low-light clades LLI, LLII, LLIII, LLIV, LLV & LLVI, based on their 16S-23S intergenic transcribed ribosomal sequence signatures (Rocap et al., 2003). The LLIV clade, including strain MIT9313, falls near the base of the *Prochlorococcus* radiation. First isolated in 1993 from 135 m into the north Atlantic, Clade LLIV are characterized by preference for low light,  and are most populous between 120 m and 200 m depths (Partensky et al., 1999). LLIV members are also the only strains found in OMZ that have been, to date, cultured, although two as yet uncultured clades, LLV and LLVI, have been detected in the OMZ of the eastern Tropical Pacific Ocean (Lavin et al., 2010) ADDITIONAL REFS XXXX. LLII and LLIII clades are often grouped together as the second oldest phylogenetic clades and share an affinity for low light conditions; LLIII strain SS120, the first *Prochlorococcus* strain to be characterized in 1988, was isolated 135 m into the Sargasso Sea. The LLI clade includes NATL1A and NATL2A, which prefer moderate irradiances found between 30 and 100 m. HL clades are the most recently evolved and despite reduced genome size in comparison to LL ecotypes, have adapted to life at higher irradiances. HLII is the most abundant group in the North Atlantic and North Pacific Oceans, often constituting over 90% of the total population (Partensky et al., 1999),  and are most numerous around 50 m depth (Partensky et al., 1999). The HLI clade, including MED4, was found at the very surface of the Mediterranean Sea, just 5 m below the surface (Partensky et al., 1999). 

*Prochlorococcus* are difficult to culture, as their reduced genomes – the smallest of any known oxyphototroph – render them partially dependent upon mutualistic heterotrophic bacteria to  detoxify reactive oxygen species (Moore et al., 2007; Morris et al., 2008). MED4, SS120 and MIT9313 have been successfully cultured in laboratories, and used to show that ecotypic classifications correspond to biochemical differences among strains. CITATONS OUR LAB and OTHERS.

Under full atmospheric [O~2~], LL clades of *Prochlorococcus* are restricted to growth under low light, in part because they suffer photoinhibition of Photosystem II (PSII) at irradiances preferred by their HL relatives. The main target of photoinhibition is the PsbA protein, which along with its homolog PsbD forms a heterodimer binding all redox active components of the PSII reaction center core (Barber, 2012). Under light-induced damage, new PsbA cannot be incorporated until the damaged PsbA is removed.  Thus, degradation of PsbA, mediated largely by FtsH proteases (Mann et al., 2000; Adam et al., 2005), is a rate-limiting step in recovery from photoinhibition (Kanervo et al., 1993). FtsH is a membrane-bound (Sacharz et al., 2015)(Zak et al., 2001), ATP, zinc metalloprotease that extracts and degrades membrane-bound proteins for degradation (Chiba et al., 2002) (Yoshioka-Nishimura and Yamamoto, 2014). In the model cyanobacterium *Synechocystis* sp. PCC 6803 4 isoforms of FtsH proteins are encoded by the slr1390, slr0228, slr1604 and sll1463 genes.  In *Synechocystis*, Slr0228 and slr1604 form a hetero-hexameric ring in the thylakoid membrane with a central pore through which proteins are translocated as they are digested, a process key for PSII repair (Adam et al., 2005; Boehm et al., 2012) (Nixon et al. (2010)) (Komenda et al., 2007), when photoinactivation exposes an N-terminal “docking site” for FtsH to initiate degradation. In contrast, slr1390 and slr1604 are found in both plasma and thylakoid membranes (Pisareva et al., 2007; Sacharz et al., 2015), are essential for cell viability (Mann et al., 2000), and implicated in PSI biogenesis (AMANDA REF from ALGATECH).

*Prochlorococcus* genomes encode 4 FtsH proteins, henceforth referred to as FtsH1-4, homologous to the *Synechocystis* FtsH isofoms, and with presumably parallel functions (Table 1).  Upon a shift to higher light HLIII MED4 upregulated expression of FtsH1 and FtsH2 (Bonisteel et al. (2018)), homologs to *Synechocystis* slr0228 and slr1604, implicated in PSII repair. In contrast representative LLIII strain MIT9313 shows less overall expression of the FtsH proteases, and thus has fewer FtsH serving each photosystem.  Furthermore MIT9313 expressed primarily FtsH3 and expression did not increase in response to light stress. Through adaptation to steady low light, LLIII **Prochlorococcus* allocate resources to other processes. 

Table 1. FtsH protease homologs amongst the model cyanobacterium Synechocystis sp. PCC6803 and Prochlorococcus marinus. Protein homology determined by multiple sequence alignment with MUSCLE followed by construction of maximum likelihood phylogenetic tree using 1000 bootstrap replicates in MEGAX

Organism	Protein Homolog			
Prochlorococcus marinus	FtsH1	FtsH2	FtsH3	FtsH4
Synechocystis sp. PCC6803	Slr0228	Slr1604	Sll1463	Slr1390
k
Function


\newpage

# Materials and methods {.unnumbered}

## MetaProteomics {.unnumbered}

The OceansMap Protein Portal (OPP; https://proteinportal.whoi.edu/) is an open access online data repository (Woods Hole Oceanographic Institute, WHOI) of mass spectroscopy data on marine microbial peptides, sampled from various depths and locations worldwide. We screened a subset of the OPP for proteins annotated as from *Prochlorococcus* strains, to identify differential strategies employed by strains living at varying depths and oxygen levels within the marine water column.  We focussed on proteins mediating photosynthesis and protein metabolism. The samples for metaproteomic analyses were collected from 12 locations in the tropical North Pacific ocean along 150 W from 18 N of the equator between October 1, 2011 and October 25, 2011 during the voyage of the R/V Kilo Moana MetZyme cruise KM1128 (https://www.rvdata.us/search/cruise/KM1128; original datasets in the Biological and Chemical Oceanography Data Management Office repository; https://www.bco-dmo.org/project/2236). 

Collection and treatment of protein samples were performed following Saito et al., (2014, 2015) XXXXADD to ZoteroXXXX . Briefly, samples of seawater from depths of 20, 40, 50, 60, 70, 80, 90, 120, 150, 200, 250, 300, 380, 400, 500, 550, 600, and 800 m below the ocean surface were pumped through a 0.2 µm filter, preserved in RNAlater and frozen at -80°C until extraction. Proteins were extracted from the filter in an SDS-based detergent, embedded in tube gel, alkylated and reduced prior to in-gel trypsin digestion. Peptide spectra were generated using a Q-Exactive Orbitrap Mass Spectrometer, searched in the SEQUEST CITATIONXXXX and labelled with the most likely protein and species annotation from Uniprot.  Oxygen levels at the location of sampling were recorded. 

## *Prochlorococcus* culturing and experimental design {.unnumbered}

Three xenic cultures of *P. marinus* were obtained from Bigelow Labs, NCMA
Maine, US (REFERENCE). MED4 (CCMP1986) is from High Light-adapted (HLI) clade; SS120
(CCMP1375) is from Low Light-adapted (LLIII) clade; and MIT9313 (CCMP2773)
is from Low Light-adapted (LLIV) clade. Cultures were
maintained in incubators set to 22°C with a light/dark cycle of 12 h, with illumination from XXXXX. The PAR level for maintenance cultures reflected  Photosynthetically Active Radiation (PAR) in the source
niche of the ecotype; MED4, of 160 µmol photons m^-2^ s^-1^; SS120 and
MIT9313 at 30 µmol photons m^-2^ s^-1^. To maintain active growth all strains were
transferred weekly with XX to YY dilution with Pro99 media
[@mooreCulturingMarineCyanobacterium2007] prepared with autoclaved artificial
seawater (National
Center for Marine Algae and Microbiota (NCMA) protocol by combining salt
solution I and salt solution II using the enriched artificial seawater
(ESAW) recipe.  REPLACe with CITATION OF NCMA protocol)XXXX

Controlled growth experiments were performed using PSI Multicultivators (Figure \@ref(fig:LabeledMC); MCMIX-OD orMC1000-OD, Brno, Czech Republic ).  Each multicultivator individually controls 8 tubes at a common temperature of 22°C.  Each tube containing 70 mL of Pro99 media was inoculated with 10 mL of growing maintenance culture. In a factorial matrix design, each tube was then subject to an individual combination of sinusoidal photoperiod (4, 8, 12, 16  h);  reaching a peak PAR (30, 90, 180 µmol photons m^-2^ s^-1^), with defined spectral bandwidth (AA, BB, CC, DD nm).  [O~2~] levels (2.5 µM, 25 µM, 250 µM) were imposed by bubbling tubes with varying ratios of air and
Nitrogen (N~2~), with consistent 0.05% of Carbon Dioxide (CO~2~)
gas, delivered through a 0.2 μm sterile microfilter via a G400 gas mixing system XXXXGIVE SOURCEXXXX. [O~2~] in situ was verified using oxygen optodes (PyroScience,
Germany) inserted into tubes for
real-time measurements, with a temperature probe in
the aquarium of the bioreactor to correct [O~2~] measures for temperature
fluctuations. In addition, the Pyroscience software corrected [O~2~] based on the
salinity of the media (32 ppt). The flow rate of the gas mixture was controlled, but variations in bubbling speed, PAR and culture density affected the [O~2~] achieved in each tube. A low [O~2~] of 0.5 µM - 5 µM (reported as 2.5 µM hereafter),  was achieved by sparging with a gas mixture containing 99.95% N~2~ and 0.05%
CO~2~. An intermediate [O~2~] of 10 - 25 µM (reported hereafter as 25 µM) was achieved by sparging with a gas mixture containing 98.95% N~2~, 0.05% CO~2~ and 1% O~2~. A high O~2~ of 200 µM - 280 µM (reported hereafter as 250 µM) was achieved by sparging with lab air (78% N~2~, 21% O~2~, 1% Ar and 0.05% CO~2~).

The full crossing of all factor levels would yield 4 x 3 x 4 x 3 = 144 treatments, x 3 strains for 432 possible combinations. Consistent absence of growth of some strains under some levels of photoperiod, PAR, or [O~2~] meant we completed 291 growth factor treatment combinations. 

In situ XXXXItalicsXXX measurements of Optical Density (OD) 680 nm, a proxy for cell suspension density, cell size dependent scatter and cell chlorophyll content;  and OD 720 nm, a proxy for cell suspension density and cell size dependent scatter, were recorded every 5 minutes over least 8 to 14 days,  depending on the
duration of the lag phase, if any. All data from the Multicultivator were saved
as comma separated values file  (CITATION of FILES LOCATION).

PAR of 180, 90 or 30 µmol photons m^-2^ s^-1^, and spectral wavebands (white LED
full spectrum, 660 nm, 530 nm, and 450 nm) were chosen to approximate light
levels and spectral colors spanning the vertical ocean water column, from
near-surface to the lower euphotic zone depths. Photoperiods were chosen to approximate diel cycles characteristic of current and hypothetical future
niches of *P. marinus*; 16 h represents
temperate (45°N) summer at the ocean surface; 12 h for equatorial (0°N)
ocean surface or temperate (45°N) spring and fall ocean surface or
temperate (45°N) summer at deeper ocean depths; 8 h for temperate (45°N)
winter at the surface or at temperate (45°N) spring and fall at depth
and equatorial (0°N) deep ocean depths; and 4 h for temperate (45°N)
winter or deep ocean depths during temperate (45°N) spring and fall.


```{r LabeledMC, fig.cap = LabeledMC_cap , fig.height = 8, fig.width = 6, echo = FALSE}

LabeledMC_cap <- glue("**PSI MCMIX-OD Multicultivator.**",
                      "Spectral wavebands and light levels are individually controlled for each culture tube.",
                      "Real time Optical Density (OD) measurements eliminate intrusive subsampling of sterile cultures.",
                      "The temperature of culture tubes are collectively controlled via heating or cooling of the aquarium water.",
                      "Gas with specific oxygen concentrations is bubbled through a humidifier and passed through a 0.2 um filter.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "LabeledMC.png"))
```

\newpage

## MetaProteomics bioinformatic analyses and limitations  {.unnumbered}

	Metaproteomic datasets were obtained from the KM1128 entry in the BCO-DMO database (https://www.bco-dmo.org/deployment/59053) accessed via the OPP in June 2019. Datasets contained:
	i) Peptide sequences and sample identification (ID) number;
	ii) Sample ID number, station, depth in meters below the surface the sample was collected at, best-hit BLASTP protein and species annotation and the corresponding Uniprot Entry number for the identified proteins;
	iii) Sample station depth and [O~2].  
	The depth and [O~2] were joined to peptide sequence and BLASTP annotations by ID number, depth and station using the functions in the Tidyverse package (https://www.tidyverse.org/) in RStudio v1.2.5019 (https://rstudio.com/)  (LOCATION OF CODE ON GITHUBO. The resulting merged dataset was filtered for *Prochlorococcus* peptides, detected from 0 to 300 m below the surface, annotated as a subunit of *Prochlorococcus* chlorophyll binding proteins (Pcb); Photosystem II (PSII); Cytochrome b6f (Cytb6f);  Photosystem I (PSI); NADPH Dehydrogenase (NDH); Terminal Oxidase (PTOX); plastocyanin (PC); ferredoxin (Fd); Ribulose-1,5-bisphosphate oxygenase (RUBISCO);  ATP Synthase;  FtsH proteases (FtsH) or ribosomes. 
	Detected peptides were re-annotated for consistency and labelled according to strain, clade, subunit and protein complex (where possible). Full protein sequences corresponding to detected proteins were obtained from UniProt (https://www.uniprot.org/) and analyzed in Molecular Evolution and Genetic Analyses X (MEGAX) software  (https://www.megasoftware.net/). Sequences for each protein and/or subunits for each of the thirteen Prochlorococcus strains identified in the dataset were aligned with MUSCLE using UPGMA cluster method and a lambda of 24 with a -2.9 gap open penalty and 1.20 hydrophobicity multiplier.  Overall mean pairwise distance between protein sequences was determined using bootstrap variance estimation methods. Maximum likelihood phylogenetic trees were assembled using 1000 bootstrap replications with a 95% site coverage cut off. *Prochlorococcus* FtsH isoform identities were specified by phylogenetic comparison to the characterized FtsH proteases of Synechocystis sp. PCC6803 (CITATONS).  Data for each strain was plotted against depth and  [O~2] and sampling station.
	
	Graphs depict the detection of mass-spectrally detected peptides at varying depths and [O~2] in the water column for  *Prochlorococcus* strains and clades identified in the dataset. When assessing the presence of a particular protein complex, all peptides belonging to all subunits of the complex were included to give the greatest number of data points. 
	Although not a direct measurement, it is assumed that light intensity is inversely correlated with depth. 
	As this data was acquired on a discovery mission rather than through targeted peptide approaches, it is difficult to discern accuracies of strain annotations, particularly as the proteins of interest in this study are highly conserved in sequence across strain. We are confident in clade and ecotypic classification for each protein examined, although exact strain annotation may be ambiguous.
Another caveat to interpretation of this data is the “peptide detection bias” inherent to mass spectrometry: certain peptides are more easily detected and thus dominate the spectra over less easily detected peptides. The data is also limited by the number and nature of protein spectra in the SEQUEST database: a peptide sequence cannot be determined unless there is already a known spectrum for that peptide in the SEQUEST database, hence some peptides of interest may not be identifiable. Therefore, the absence of a particular does not necessarily rule out its presence overall.  Furthermore, a peptide must be detected above a certain threshold abundance in order to be considered an accurate ‘hit’. Therefore, the ‘absence’ of a particular protein structure is here interpreted as low in quantity, while its presence is interpreted as higher in quantity.

## Growth data management and analysis {.unnumbered}
Data files saved form from the Multicultivator software were imported into R-Studio (CITATION) for data management [@R-tidyverse], growth rate calculations CITATION, comparisons of model fits CITATION, and generation of
figures`ggplot2` CITATION. The chlorophyll proxy
optical density (OD680 - OD720;  ΔOD) was used to determine the chlorophyll-specific exponential growth
rate (µ, h^-1^) for each treatment combination.  We first we used a rolling mean from the RStudio `zoo` package [@R-zoo] to calculate the average ΔOD data over a 1-hour window, to lower influence of outlier points XXXXother trimmingXXXXX using a Levenberg-Marquardt algorithm
[@bellaviaLevenbergMarquardtMethod2018] modification of the non-linear
least squares,  using the R package `minpack.lm`
[@R-minpack.lm], to fit a logistic
equation \@ref(eq:GrowthRate) where ΔOD~max~ is maximum ΔOD, ΔOD~min~ is minimum ΔOD, t is time
duration over the growth trajectory. 
OTHER MOREXXXXXX

```{=tex}
\begin{equation}
  µ = \frac{ΔOD_{max} × ΔOD_{min} × exp^{(µ × t)}}{ΔOD_{max} + (ΔOD_{min} × exp^{((µ × t) - 1)})}
  (\#eq:GrowthRate)
\end{equation}
```

CITAtIONS of github locations of code etc. XXXXX

### Generalized additive model of growth responses {.unnumbered}

%%%%%% Taken from my thesis word for word, need to reword and shorten %%%%

A Generalized Additive Model (GAM) was applied to examine the
relationship between chlorophyll-specific µ, h^-1^, for each [O~2~] level, across the blue and red spectral wavebands, photoperiods and PAR levels for each *P.marinus* strain in this study. The gam function from the R package `mgcv`
[@R-mgcv] was used to model the growth rate with smoothing terms to
indicate the 90, 50 and 10% quantiles for growth rate across the levels of factors. Only data  XXXXXWHAT DATAXXXX BE SPECIFICXXXX below a standard error tolerance of 30% of the fit was used in the model. Our priority
was studying the effect of blue light on growth trends, since blue light is the most ecologically relevant spectral waveband for
deep ocean niches.  We included analyses of responses of red light, which is not ecophysiologically relevant but might prove mechanistically informative (CITE MURPHY ETC)

### Estimation of Photosynthetically Usable Radiation

XXXXTEXT NEEDEDXXXX
```{r PurParPlot, fig.cap = PurParPlot_cap , fig.height = 20, fig.width = 10, echo = FALSE}

PurParPlot_cap <- glue("**The absorbed Photosynthetically Usable Radiation (PUR) (µmol photons m^-2^ s^-1^) vs. the Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^).**",
                       "The correlation between PAR, plotted on the x-axis and absorbed PUR, plotted on the y-axis, are colored for each growth spectral waveband; blue for 450 nm, red for 660 nm and black for white LED full spectrum light.",
                       "The grey dashed line represents a hypothetical one to one correlation.",
                       "**A.** is *Prochlorococcus marinus*  MED4.",
                       "**B.** is *Prochlorococcus marinus*  SS120.",
                       "**C.** is *Prochlorococcus marinus*  MIT9313.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PurParPlot.png"))
```


\newpage

# Results and Discussion {.unnumbered}

%%%% OceanProteinPortal data show complexes in environment  %%%


```{r ProchloroPhotosynthDepthO2, fig.cap = ProchloroPhotosynthDepthO2_cap , fig.height = 20, fig.width = 10, echo = FALSE}

ProchloroPhotosynthDepthO2_cap <- glue("**Ocean detection of *Prochlorococcus marinus* photosynthesis complexes.**", 
                       "Protein detections are plotted vs. O~2~ (µM) (X axis) and depth (m) (Y axis) at sample origin.",
                       "Rows separate data annotated as from *Prochlorococcus marinus* strains MED4, SS120 and MIT9313.",
                       "Columns show detections of proteins annotated as  Photosystem II (PSII), Cytochromeb6f complex (Cytb6f), Photosystem I (PSI), ATP Synthase or Ribulose-1,5-bisphosphate oxygenase carboxylase (RUBISCO).",
                       "Culture growth experimental conditions indicated by horizontal grey lines for depths approximating Photosynthetically Active Radiation (µmol photons m^-2^ s^-1^) and vertical grey lines for O~2~ (µM).",
                       "Point colours identify *Prochlorococcus* clade assignments",
                       "Data from OceanProteinPortal (https://www.oceanproteinportal.org/).",
 .sep = " ")

knitr::include_graphics(file.path(FiguresInOceanPro, "ProchloroPhotosynthDepthO2.png"))
```

\newpage

\newpage

```{r MED4PAR, fig.cap = MED4PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MED4PAR_cap <- glue("**Chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* MED4 (High Light (HLI) near surface clade) vs. photoperiod (h).  **",  
                    
                    "3 levels of growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) are in columns;  3 levels of imposed growth dissolved O~2~ concentrations (µM) are in rows.",
                    "Colors represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MED4PAR.png"))
```

\newpage

```{r SS120PAR, fig.cap = SS120PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

SS120PAR_cap <- glue("**Chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* SS120 (Low Light (LLIII) deep ocean clade) vs. photoperiod (h). **",  
                     
                    "3 levels of growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) are in columns;  3 levels of imposed growth dissolved O~2~ concentrations (µM) are in rows.",
                    "Colors represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",
                    
                       .sep = " ")
                       
knitr::include_graphics(file.path(FiguresIn, "SS120PAR.png"))

```

\newpage

```{r MIT9313PAR, fig.cap = MIT9313PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MIT9313PAR_cap <- glue("**Chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* MIT9313 (Low Light (LLIV) deep ocean clade) vs. photoperiod (h). **",  
                       
                    "3 levels of growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) are in columns;  3 levels of imposed growth dissolved O~2~ concentrations (µM) are in rows.",
                    "Colors represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",

                       .sep = " ")
                       
knitr::include_graphics(file.path(FiguresIn, "MIT9313PAR.png"))

```


```{r MED4GAM,  fig.cap = MED4GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

MED4GAM_cap <- glue("**A contour plot of a Generalized Additive Model (GAM) representing the chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* MED4 grown under 660 nm (red) and 450 nm (blue) light.**",
                    "The x-axis is photoperiod (h).",
                    "The y-axis is actinic Photosynethetic Active Radiation (PAR) level (µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 250 µM of O~2~ and blue light.",
                    "**D.** represents the model under 25 µM of O~2~ and blue light.",
                    "The legend represents a colour gradient of growth rate from no growth (white) to 1 d^-1^ (dark blue).",
                    "The labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_MED4_Oxy_.png"))


```

\newpage


```{r SS120GAM,  fig.cap = SS120GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

SS120GAM_cap <- glue("**A contour plot of a Generalized Additive Model (GAM) representing the chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* SS120 grown under 660 nm (red) and 450 nm (blue) light.**",
                    "The x-axis is photoperiod (h).",
                    "The y-axis is actinic Photosynethetic Active Radiation (PAR) level (µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 250 µM of O~2~ and blue light.",
                    "**D.** represents the model under 25 µM of O~2~ and blue light.",
                    "The legend represents a colour gradient of growth rate from no growth (white) to 1 d^-1^ (dark blue).",
                    "The labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_SS120_Oxy_.png"))


```

\newpage

```{r MIT9313GAM,  fig.cap = MIT9313GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

MIT9313GAM_cap <- glue("**A contour plot of a Generalized Additive Model (GAM) representing the chlorophyll proxy growth rate (d^-1^) for *Prochlorococcus marinus* MIT9313 grown under 660 nm (red) and 450 nm (blue) light.**",
                    "The x-axis is photoperiod (h).",
                    "The y-axis is actinic Photosynthetically Active Radiation (PAR) level (µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 2.5 µM of O~2~ and red light.",
                    "**D.** represents the model under 250 µM of O~2~ and blue light.",
                    "**E.** represents the model under 25 µM of O~2~ and blue light.",
                    "**F.** represents the model under 2.5 µM of O~2~ and blue light.",
                    "The legend represents a colour gradient of growth rate from no growth (white) to 1 d^-1^ (dark blue).",
                    "The labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_MIT9313_Oxy_.png"))


```

\newpage

%%%% Insert PURFits figures  %%%

```{r AllStrainPurFitsPlots,  fig.cap = AllStrainPurFitsPlots_cap, fig.height = 10, fig.width = 10, echo = FALSE}

AllStrainPurFitsPlots_cap <- glue("** The chlorophyll proxy growth rate (d^-1^) vs. the Photosynthetic Usable Radiation (PUR) (µmol photons m^-2^ d^-1^).**",
                    "The x-axis is the Photosynthetic Usable Radiation (PUR) (µmol photons m^-2^ s^-1^) .",
                    "The y-axis is the chlorophyll specific growth rate (d^-1^).",
                    "**A.** The High Light I (HLI) ecotype MED4.  under 250 µM of O~2~ and red light.",
                    "**B.** The Low Light II/II (LLII/III) ecotype SS120. under 25 µM of O~2~ and red light.",
                    "**C.** The Low Light IV (LLIV) ecotype MIT9313. under 2.5 µM of O~2~ and red light.",
                     "Shapes show the imposed photoperiod (h); hollow inverted triangle for 4 h, solid diamond for 8 h, solid circle for 12 h, solid upright triangle for 16 h.",
                      "Symbol colours show the spectral waveband for growth; black for white LED light (full spectrum), red for red light (660 nm), and blue for blue light (450 nm).",
                      "Large shapes show mean or single determinations of growth rate from logistic curve fits; small shapes show values for replicate determinations, if any.",
                    "The 3 levels of imposed growth dissolved O~2~ concentrations (µM) are in rows.",
                       .sep = " ")


knitr::include_graphics(file.path(FiguresIn, "AllStrainPurFitsPlots.png"))

```


\newpage

%%%% OceanFtsHRibosomeProteins figure from OceanPortal repository  %%%


```{r ProchloroProteinMetabDepthO2, fig.cap = ProchloroProteinMetabDepthO2_cap , fig.height = 20, fig.width = 10, echo = FALSE}

ProchloroProteinMetabDepthO2_cap <- glue("**Ocean detection of *Prochlorococcus marinus* protein metabolism complexes.**", 
                       "Protein detections are plotted vs. O~2~ (µM) (X axis) and depth (m) (Y axis) at sample origin.",
                       "Rows separate data annotated as from *Prochlorococcus marinus* strains MED4, SS120 and MIT9313.",
                       "Columns show detections of proteins annotated as FtsH Protease Complexes (FtsH1, FtsH2, FtsH3) or the Ribosome).",
                       "Culture growth experimental conditions indicated by horizontal grey lines for depths approximating Photosynthetically Active Radiation (µmol photons m^-2^ s^-1^) and vertical grey lines for O~2~ (µM).",
                       "Point colours identify *Prochlorococcus* clade assignments",
                       "Data from OceanProteinPortal (https://www.oceanproteinportal.org/).",
 .sep = " ")



knitr::include_graphics(file.path(FiguresInOceanPro, "ProchloroMetabDepthO2.png"))
```






```{r MED4ProteinAnnotated, fig.cap = MED4ProteinAnnotated_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MED4ProteinAnnotated_cap <- glue("**fmole target protein per ug total protein for *Prochlorococcus marinus* MED4 (High Light (HLI) near surface clade).  **",  
                    
                    "Growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) and spectral wavelength are in rows;  2 levels of imposed growth dissolved O~2~ concentrations (µM) are in columns.",
                    "Numbers over each bar are fmole/ug",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MED4ProteinAnnotated.png"))
```

\newpage



```{r MIT9313ProteinAnmotated, fig.cap = MIT9313ProteinAnmotated_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MIT9313ProteinAmnotated_cap <- glue("**fmole target protein per ug total protein for *Prochlorococcus marinus* MIT9313 (Low Light (LLIV) deep ocean clade).  **",  
                    
                    "Growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) and spectral wavelength are in rows;  2 levels of imposed growth dissolved O~2~ concentrations (µM) are in columns.",
                    "Numbers over each bar are fmole/ug",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MIT9313ProteinAnnotated.png"))
```

\newpage


# Supplemental {.unnumbered}


# Acknowledgements
Jonah Sheinin assisted with growth of some cultures for protein analyses, while Carlie Barnhill assisted with code for import of multicultivator growth data files.


# Funding
To Be Entered through PLoSONe system and deleted here
Czech Academy of Science visiting fellowship (DAC)
Canada Research Chair in Phytoplankton Ecophysiology (DAC)
Natural Sciences and Engineering Research Council of Canada, 'Latitude and Light' (DAC)
Canada Foundation for Innovation (DAC)
New Brunswick Foundation for Innovation (DAC)
Rice Graduate Fellowship (MS)

# References {.unnumbered}