---
title: "*Prochlorococcus marinus* responses to light and oxygen" 
author:
  - name: 'Mireille Savoie'
    institute: mta
    main: true
    email: msavoie@mta.ca
  - name: Aurora Mattison
    institute: [mta, ubc]
  - name: Julie Nadeau
    institute: mta
  - name: Laurel Genge
    institute: mta
  - name: Sylwia Śliwińska-Wilczewska
    institute: [mta, ug]
  - name: Maximilian Berthold
    institute: mta
  - name: Naaman M. Omar
    institute: mta
  - name: Ondřej Prášil
    institute: [mta, algatech]
    main: false
  - name: Amanda M. Cockshutt
    institute: [mta, sfx]
    main: false
  - name: Douglas A. Campbell
    institute: mta
    affil: 1
    main: false
    orcid: '0000-0001-8996-5463'
    email: 'dcampbel@mta.ca'
affiliation:
  - num: 1
    address: Departments of Biology and Chemistry and Biochemistry, Mount Allison University
institute:
  - mta: 'Department of Biology, Mount Allison University, 63B York St., Sackville NB, Canada, E4L 1G7.'
  - ubc: 'Department of Biology, University of British Columbia'
  - sfx: 'Department of Chemistry, St. Frances Xavier University'
  - algatech: 'Institute of Microbiology, Center Algatech, Laboratory of Photosynthesis, Novohradska 237, Trebon, CZ 37981, Czech Republic.'
  - ug: 'Institute of Oceanography, University of Gdansk, 46 Piłsudskiego St, Gdynia, Poland'
date: "`r format(Sys.Date())`"
output:
  bookdown::html_document2:
    code_folding: show
    keep_md: yes
    toc: TRUE
    toc_float: TRUE
    toc_depth: 6
    fig_caption: yes
    pandoc_args: 
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
  bookdown::word_document2:
    code_folding: show
    reference_docx: KnitTemplate.docx
    keep_md: yes
    toc: FALSE
    toc_depth: 6
    fig_caption: yes
    pandoc_args: 
      - '--lua-filter=scholarly-metadata.lua'
      - '--lua-filter=author-info-blocks.lua'
always_allow_html: true
bibliography: [Manuscript_O2.bib, RPackages.bib]
csl: plos-one.csl
editor_options: 
  markdown: 
    wrap: 72
---

```{css, echo=FALSE}
p.caption {
  font-size: 12px;
}
```

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE, message = FALSE, warning = FALSE)
knitr::opts_chunk$set(fig.path='Figs/')
```

```{r load libraries, include = FALSE}
# libraries; Note check actual dependencies
# Naaman add packages?
library(knitr)
library(tidyverse)
library(broom)
library(grid)
library(bookdown)
library(glue)
library(cowplot)
library(ggpubr)

library(png)
library(cowplot)

knitr::write_bib(c("knitr", "tidyverse", "lubridate", "ggplot2", "broom", "grid", "bookdown", "glue", "cowplot", "ggpubr", "ggh4x", "png", "Studio", "zoo", "photobiology", "photobiologyWavebands", "data.table", "googlesheets4",  "mgcv", "minpack.lm",  "egg", 'RPackages.bib'))

```

```{r set project variables, include = FALSE}
#"..", takes up a level in the directory path
Project <- "PICO"
DataIn <- file.path("..", "Data", "CleanData")
FiguresIn <- file.path("..","Output", "Figures")
FigureInO2Analy <- file.path(  "..", "..", "O2Analyses", "Output")

```

```{r read ProcessFile, include = FALSE}

 PURPARFits <- readRDS(file.path(DataIn, "PICO_Merged_GrowthFitsPURPAR.Rds"))
 
```


# Abstract {.unnumbered}


\newpage

# Introduction {.unnumbered}

## *Prochlorococcus marinus* diversity {.unnumbered}
*Prochlorococcus*, a genus of Cyanobacteria, is the smallest known photosynthetic prokaryote, with cell diameters ranging from 0.5 to 0.7 µm [@chisholmProchlorococcusMarinusNov1992]. Despite small cell size, *P. marinus* contribute 13 to 48% of net primary production in oligotrophic oceans, corresponding to about 30% of global oxygen production [@partenskyProchlorococcusMarinePhotosynthetic1999]. *Prochlorococcus marinus* growth is currently limited to between latitudes of 40°N to 40°S in open ocean waters, from surface to 300 m depth, thus spanning 3 orders of magnitudes of light irradiance [@partenskyProchlorococcusMarinePhotosynthetic1999; @chisholmProchlorococcusMarinusNov1992 ]. 

*Prochlorococcus marinus* is functionally differentiated from other cyanobacteria primarily by the absence of phycobilisomes [@chisholmProchlorococcusMarinusNov1992]. Instead, *Prochlorococcus*  uses intra-membrane Prochlorophyte chlorophyll binding (Pcb) proteins binding divinyl chlorophyll *a* and *b*, and α-carotene, as major light harvesting complexes [@goerickePigmentsProchlorococcusMarinus1992]; [@larocheIndependentEvolutionProchlorophyte1996], with diversity in the number of genes encoding Pcb, expression of Pcb under changing light [@garczarekMultiplicationAntennaGenes2000; @rocapGenomeDivergenceTwo2003], and in thylakoid organization [@partenskyProchlorococcusMarinePhotosynthetic1999].  The genes encoding much of the photosynthetic apparatus are highly conserved in *Prochlorococcus* [@kettlerPatternsImplicationsGene2007], though the photosynthetic efficiency may vary among clades [@hessPhotosyntheticApparatusProchlorococcus2001; @moorePhotophysiologyMarineCyanobacterium1999].

*Prochlorococcus marinus* comprises many strains, organized into clades, defined by 16S-23S intergenic transcribed ribosomal sequence signatures [@rocapGenomeDivergenceTwo2003]. The clades inhabit distinct ecological niches [@moorePhysiologyMolecularPhylogeny1998], originally defined by High-Light (HL) or Low-Light (LL).  Of at least 12 known *Prochlorococcus* genetic clades only 5 to date have cultured representatives; HLI, HLII, LLI, LLII/III and LIV [@billerProchlorococcusStructureFunction2015].

Current niches of *P. marinus* strains span ocean water columns [@veldhuisVerticalDistributionPigment1990; @chisholmNovelFreelivingProchlorophyte1988; @partenskyProchlorococcusMarinePhotosynthetic1999] and extend into regions with low dissolved oxygen concentrations [@partenskyProchlorococcusAdvantagesLimits2010; @goerickeNovelNicheProchlorococcus2000; @lavinNovelLineagesProchlorococcus2010, @ulloaCyanobacteriumProchlorococcusHas2021; @garcia-robledoCrypticOxygenCycling2017]. 


Low-Light clades thrive in deeper ocean waters, extending beyond 200 m in depth [@partenskyProchlorococcusMarinePhotosynthetic1999], where only ~1% of the surface irradiance penetrates, primarily in the blue (450 nm) to green (520 nm) spectral range [@holtropVibrationalModesWater2021]. Clade LLI includes cultured strain NATL2A, which prefers moderate irradiance typically between 30 and 100 m depth. Clades LLII and LLIII, including cultured strain SS120, are grouped together as the second oldest phylogenetic lineage diversifying in the *P. marinus* radiation, with an affinity for low light. Clade LLIV, including cultured strain MIT9313, falls near the base of the *Prochlorococcus* radiation, and is characterized by preference for low light, typical of depths from 120 m to 200 m [@partenskyProchlorococcusMarinePhotosynthetic1999]. LLIV members are, as yet, the only cultured strains to have been found in Oxygen Minimum Zones (OMZ).  Some, as yet, uncultured *P. marinus* strains in clades LLV and LLVI also thrive in OMZ of the subtropical Atlantic and Pacific Oceans, where dissolved oxygen concentrations [O~2~] can be less than 20 µM [@ulloaCyanobacteriumProchlorococcusHas2021; @lavinNovelLineagesProchlorococcus2010; @goerickeNovelNicheProchlorococcus2000; @garcia-robledoCrypticOxygenCycling2017; @johnsonEnergeticsGrowthKinetics1999]. *Prochlorococcus marinus* LL ecotypes may dominate the phytoplankton within OMZ [@lavinNovelLineagesProchlorococcus2010; @ulloaCyanobacteriumProchlorococcusHas2021; @partenskyProchlorococcusAdvantagesLimits2010], where they may be net O~2~ consumers [@partenskyComparisonPhotosyntheticPerformances2018].

HL clades are more evolved lineages, with reduced genome sizes in comparison to LL clades. High-Light clades are typically dominant picophytoplankters in near-surface, oligotrophic waters, characterized by high light levels. Clade HLI, represented by cultured strain MED4, is adapted to higher iron, and lower temperatures, and originated from 5 m depth in the Mediterranean Sea [@partenskyProchlorococcusMarinePhotosynthetic1999].  Clade HLII, adapted to higher iron, and higher temperatures, is the most abundant *P. marinus* clade in the North Atlantic and North Pacific Oceans, often constituting over 90% of the total population [@partenskyProchlorococcusMarinePhotosynthetic1999],  and are most numerous around 50 m depth [@partenskyProchlorococcusMarinePhotosynthetic1999]. Clade HLIII/IV is adapted to lower iron [@kentParallelPhylogeographyProchlorococcus2019; @johnsonNichePartitioningProchlorococcus2006; @zinserInfluenceLightTemperature2007].

*Prochlorococcus marinus* clades are nonetheless found in environments beyond their optimal habitats. HL clades inhabit depths overlapping with LL ecotypes [@westNichePartitioningProchlorococcusPopulationsStratified1999; @zinserInfluenceLightTemperature2007; @delmontLinkingPangenomesMetagenomes2018], while LL clades can occupy regions in OMZ at depths shallower than 40 m [@lavinNovelLineagesProchlorococcus2010], exploiting ambient light levels above what LL clades were thought to tolerate. 
 
## *Prochlorococcus* and changing niches {.unnumbered} 

Our changing climate is rapidly altering conditions for these specialized clades of marine picophytoplankton. Predictions indicate a net global increase of *P. marinus* cell abundances of 29%, along with  poleward latitudinal shifts of at least 10° in marine phytoplankton niches by the end of this century [@flombaumPresentFutureGlobal2013; @bartonAnthropogenicClimateChange2016] in response to warming waters, with increases in *P. marinus* of approximately 50% in the more poleward regions of their distributions.

Near the equator, photoperiod remains nearly constant at the ocean surface, approximately 12 hours (h) of daylight and 12 h of darkness throughout the year. The effective length of the photoperiod does, however attenuate with depth as dawn and dusk light at depth drops below levels needed for biological processes. As *P. marinus* potentially expands its temperature-permissive niches poleward into temperate regions  [@flombaumPresentFutureGlobal2013; @bartonAnthropogenicClimateChange2016], it will encounter more
pronounced seasonal variations in photoperiod regimes both at surface and at depth, with potentially complex effects upon viable growth niches [@liDiatomGrowthResponses2017; @prezelinDielPeriodicityPhytoplankton1992]. For example, Vaulot *et al*. @vaulotGrowthProchlorococcusPhotosynthetic1995 showed that *Prochlorococcus* replication of DNA occurs in the afternoon, while cell division occurs at night.  To our knowledge, no study has as yet addressed *P. marinus* growth responses in relation to a  range of photoperiods. 

Climate change is also rapidly changing ocean chemistry.  By the end of this century, surface ocean pH is projected to decline by 0.1 to 0.4  due to projected increases in carbon dioxide concentrations [@garcia-sotoOverviewOceanClimate2021]. Moreover, substantial changes in the global water cycle, leading to extensive changes in worldwide precipitation patterns, are affecting ocean salinity levels on a global scale, and ice melts due to rising temperatures are impacting salinity levels in the Arctic and Northwest Atlantic oceans [@leeRecognizingSalinityThreats2022]. Increasing sea temperatures are also causing decreases in [O~2~] across global oceans [@matearLongtermChangesDissolved2003], particularly toward the poles [@helmObservedDecreasesOxygen2011]. Warmer ocean waters increase stratification, and decrease oxygen solubility at the surface, which in turn decreases oxygen mixing downwards by ocean currents [@garcia-sotoOverviewOceanClimate2021]. Models predict that OMZ in the Pacific and Indian Oceans are expanding [@garcia-sotoOverviewOceanClimate2021; @buseckeDivergingFatesPacific2022]. In spite of these expanding OMZ, Busecke *et al*. [@buseckeDivergingFatesPacific2022] project that the cores of the OMZ, where the oxygen levels are lowest, may actually contract.

We used the OceansMap Protein Portal (OPP; https://proteinportal.whoi.edu/) [@saitoDevelopmentOceanProtein2021] to analyze the distribution of proteins from clades of *P. marinus* in samples taken across a range of [O~2~] and depth, which in turn correlates to peak light at the site of sampling. In parallel we analyzed the growth and physiological responses of representative strains from three clades of *P. marinus* under a matrix of [O~2~], light levels, spectral waveband ranges, and photoperiods, to approximate eco-physiological conditions representative of current and hypothetical future ocean zones. *Prochlorococcus marinus* MED4, a clade HLI strain, was isolated near the ocean surface (5m depth) of the Mediterranean Sea where [O~2~] is near saturation, light levels are high and spectral bias from full solar irradiance is minimal. *Prochlorococcus marinus* SS120, a clade LLIII strain, was isolated from the Sargasso Sea at a depth of 120 m, while *P. marinus* MIT9313, a clade LLIV strain, was isolated from the North Atlantic Gulf Stream at a depth of 135 m. At these depths, light attenuation and spectral shifts occur, resulting in low blue and green light, while [O~2~] varies from near-surface saturation levels to decreased concentrations, but does not necessarily decrease systematically with depth [@billheimerOxygenSeasonalityUtilization2021]. 

Photosynthetic organisms absorb light energy within the Photosynthetically Active Radiation (PAR) range, 350 to 700 nm, for photosynthesis [@morelAvailableUsableStored1978]. Photosynthetically Usable Radiation (PUR) represents the fraction of PAR that is absorbed by the pigments of a given photosynthetic organism [@morelAvailableUsableStored1978], taking into account the specific spectral wavebands these pigments absorb. *Prochlorococcus marinus* light-harvesting complexes, [@chisholmProchlorococcusMarinusNov1992;
@goerickePigmentsProchlorococcusMarinus1992], show an absorption maxima
of 442 nm for divinyl chlorophyll *a* and 478 nm for divinyl chlorophyll *b*
[@goerickePigmentsProchlorococcusMarinus1992] allowing *P. marinus* to
efficiently harvest blue light in the 400 nm to 500 nm range [@morelAvailableUsableStored1978] prevailing in deep ocean habitats, where only blue spectral wavelengths prevail [@holtropVibrationalModesWater2021].  In *P. marinus* small cell diameters, from 0.5 to 0.7 µm [@chisholmProchlorococcusMarinusNov1992], and simple cell structures, minimize the complication of pigment package effect or intracellular self-shading [@morelProchlorococcusSynechococcusComparative1993]. Given the different spectral light regimes typical of the niches of different ecotypes, expressing growth rates in terms of cumulative diel PUR might simplify different photoperiods, spectral bands, and PAR levels into a common parameter, making growth response comparisons across strains and different oxygen levels more accessible.  We also aimed to detect whether growth responses are driven simply by cumulative diel PUR, or whether specific photoperiods, spectral bands or PAR levels have independent, albeit interacting, effects on growth. We therefore analyzed growth rates in terms of cumulative diel PUR.

We discuss our findings in relation to analyses of genomic sequences CITATIONS across clades of *P. marinus*, showing that differences in the presence of genes encoding oxygen-dependent enzymes, and DNA repair enzymes, can help explain differential growth responses of strains under the matrix of light and [O~2~] conditions.

\newpage


# Materials and methods {.unnumbered}

## MetaProteomics {.unnumbered}

The OceansMap Protein Portal is an open access online data repository (Woods Hole Oceanographic Institute, WHOI) of mass spectroscopy data on marine microbial peptides, sampled from various depths and locations worldwide [@saitoDevelopmentOceanProtein2021]. We screened a subset of the OPP for proteins annotated as from *Prochlorococcus* strains, to identify differential strategies employed by strains living at varying depths and oxygen levels within the marine water column.  We focussed on proteins mediating photosynthesis and protein metabolism. The samples for metaproteomic analyses were collected from 12 locations in the tropical North Pacific ocean along 150 W from 18 N of the equator between October 1, 2011 and October 25, 2011 during the voyage of the R/V Kilo Moana MetZyme cruise KM1128 (https://www.rvdata.us/search/cruise/KM1128; original datasets in the Biological and Chemical Oceanography Data Management Office repository; https://www.bco-dmo.org/project/2236). 

Collection and treatment of protein samples were performed by Saito *et al*. [@saitoMultipleNutrientStresses2014; @saitoNeedlesBlueSea2015]. Briefly, samples of seawater from depths of 20, 40, 50, 60, 70, 80, 90, 120, 150, 200, 250, 300, 380, 400, 500, 550, 600, and 800 m below the ocean surface were pumped through a 0.2 µm filter, preserved in RNAlater and frozen at -80°C until extraction. Proteins were extracted from the filter in an SDS-based detergent, embedded in tube gel, alkylated and reduced prior to in-gel trypsin digestion. Peptide spectra were generated using a Q-Exactive Orbitrap Mass Spectrometer, searched in the SEQUEST CITATIONXXXX and labelled with the most likely protein and species annotation from Uniprot.  Oxygen levels at the location of sampling were recorded. 

## MetaProteomics bioinformatic analyses  {.unnumbered}

Metaproteomic datasets were obtained from the KM1128 entry in the BCO-DMO database (https://www.bco-dmo.org/deployment/59053) accessed via the OPP in June 2019. Datasets contained:
	i) Peptide sequences and sample identification (ID) number;
	ii) Sample ID number, station, depth in meters below the surface the sample was collected at, best-hit BLASTP protein and species annotation and the corresponding Uniprot Entry number for the identified proteins;
	iii) Sample station depth and [O~2~].  
	The depth and [O~2~] were joined to peptide sequence and BLASTP annotations by ID number, depth and station using the Tidyverse package [@R-tidyverse] in RStudio v1.2.5019 (https://rstudio.com/)  (LOCATION OF CODE ON GITHUB). The resulting merged dataset was filtered for those *Prochlorococcus* peptides, detected from 0 to 300 m below the surface, annotated as a subunit of *Prochlorococcus* chlorophyll binding proteins (Pcb); Photosystem II (PSII); Cytochrome b6f (Cytb6f);  Photosystem I (PSI); NADPH Dehydrogenase (NDH); Terminal Oxidase (PTOX); plastocyanin (PC); ferredoxin (Fd); Ribulose-1,5-bisphosphate oxygenase (RUBISCO);  ATP Synthase;  FtsH proteases (FtsH) or ribosomes. 
	Detected peptides were re-annotated for consistency and labelled, where feasible, according to strain, clade, subunit and protein complex. Full protein sequences corresponding to detected proteins were obtained from UniProt (https://www.uniprot.org/) and analyzed in Molecular Evolution and Genetic Analyses X (MEGAX) software  (https://www.megasoftware.net/). Sequences for proteins for each of the thirteen Prochlorococcus strains identified in the dataset were aligned with MUSCLE using UPGMA cluster method and a lambda of 24 with a -2.9 gap open penalty and 1.20 hydrophobicity multiplier.  Overall mean pairwise distance between protein sequences was determined using bootstrap variance estimation methods. Maximum likelihood phylogenetic trees were assembled using 1000 bootstrap replications with a 95% site coverage cut off. *Prochlorococcus* FtsH isoform identities, and functions, were inferred by sequence comparisons to the characterized four isoforms of FtsH protease of *Synechocystis* sp. PCC6803 (CITATONS).  Data for each strain was plotted against depth and  [O~2~] and sampling station.
	
When assessing the presence of a particular protein complex at a sampling location, all peptides belonging to all subunits of the complex were included to give the greatest number of data points. As this data was acquired on a discovery mission rather than through targeted peptide approaches, it is difficult to discern accuracies of strain assignment annotations, particularly as the proteins of interest in this study are highly conserved across strains. We are, however, confident in clade classifications for each protein examined. A caveat to interpretation of this data is the peptide detection bias inherent to mass spectrometry CITATION. The data is also limited by the number and nature of protein spectra in the SEQUEST database: a peptide sequence was not determined unless there is already a known spectrum for that peptide in the SEQUEST database, hence some peptides of interest may not be identifiable. Furthermore, a peptide must be detected above a certain threshold abundance in order to be considered an accurate ‘hit’.

## *Prochlorococcus* culturing and experimental design {.unnumbered}

Three xenic cultures of *P. marinus* were obtained from Bigelow Labs, NCMA
Maine, US (REFERENCE). MED4 (CCMP1986) is from High-Light adapted (HLI) clade; SS120
(CCMP1375) is from Low-Light adapted (LLIII) clade; and MIT9313 (CCMP2773)
is from Low-Light adapted (LLIV) clade. Cultures were
maintained in incubators set to 22°C with a light/dark cycle of 12 h. The PAR level for maintenance cultures reflected Photosynthetically Active Radiation (PAR) in the source
niche of the ecotype; MED4, of 160 µmol photons m^-2^ s^-1^ with illumination from STANDARD Products Inc. Cool White F24T5/41K/8/HO/PS/G5/STD, 24 watts, fluorescent bulbs; SS120 and
MIT9313 at 30 µmol photons m^-2^ s^-1^ with illumination from Philips Cool White F14T5/841 Alto, 14 watts, fluorescent bulbs. To maintain active growth all strains were
transferred weekly with 1 in 5 dilution with Pro99 media
[@mooreCulturingMarineCyanobacterium2007] prepared with autoclaved artificial
seawater [@bergesEvolutionArtificialSeawater2001].

Controlled growth experiments were performed using PSI Multicultivators (Figure \@ref(fig:LabeledMC); MCMIX-OD or MC1000-OD, Brno, Czech Republic).  Each multicultivator individually controls 8 tubes at a common temperature of 22°C.  Each tube containing 70 mL of Pro99 media was inoculated with 10 mL of growing maintenance culture. In a factorial matrix design, each tube was then subject to an individual combination of sinusoidal photoperiod (4, 8, 12, 16  h);  reaching a peak PAR (30, 90, 180 µmol photons m^-2^ s^-1^), with defined spectral bandwidth (White LED, 660 nm, 450 nm).  [O~2~] levels (2.5 µM, 25 µM, 250 µM) were imposed by bubbling tubes with varying ratios of air and
Nitrogen (N~2~), with consistent 0.05% of Carbon Dioxide (CO~2~)
gas, delivered through a 0.2 μm sterile microfilter via a G400 gas mixing system (Qubit Systems Inc, Kingston, Ontario, Canada). [O~2~] in situ was verified using oxygen optodes (PyroScience,
Germany) inserted into tubes for
real-time measurements, with a temperature probe in
the aquarium of the bioreactor to correct [O~2~] measures for temperature
fluctuations. In addition, the Pyroscience software corrected [O~2~] based on the
salinity of the media (32 ppt). The flow rate of the gas mixture was controlled, but variations in bubbling speed, PAR and culture density affected the [O~2~] achieved in each tube. A low [O~2~] of 0.5 µM - 5 µM (reported as 2.5 µM hereafter),  was achieved by sparging with a gas mixture containing 99.95% N~2~ and 0.05%
CO~2~. An intermediate [O~2~] of 10 - 25 µM (reported hereafter as 25 µM) was achieved by sparging with a gas mixture containing 98.95% N~2~, 0.05% CO~2~ and 1% O~2~. A high O~2~ of 200 µM - 280 µM (reported hereafter as 250 µM) was achieved by sparging with lab air (78% N~2~, 21% O~2~, 1% Ar and 0.05% CO~2~).

The full crossing of all factor levels would yield 4 x 3 x 3 x 3 = 108 treatments, x 3 strains for 324 possible combinations. Consistent absence of growth of some strains under some levels of photoperiod, PAR, or [O~2~] meant we completed 268 growth factor treatment combinations. 

*In situ* measurements of Optical Density (OD) 680 nm, a proxy for cell suspension density, cell size dependent scatter and cell chlorophyll content;  and OD 720 nm, a proxy for cell suspension density and cell size dependent scatter, were recorded every 5 minutes over least 8 to 14 days,  depending on the
duration of the lag phase, if any. All data from the Multicultivator were saved
as comma separated values files  (https://github.com/FundyPhytoPhys/prochlorococcus_o2/tree/main/Data/RawData).

PAR of 180, 90 or 30 µmol photons m^-2^ s^-1^, and spectral wavebands (white LED
full spectrum, 660 nm, and 450 nm) were chosen to approximate light
levels and spectral colours spanning the vertical ocean water column, from
near-surface to the lower euphotic zone depths. Photoperiods were chosen to approximate diel cycles characteristic of current and hypothetical future
niches of *P. marinus*; 16 h represents
temperate (45°N) summer at the ocean surface; 12 h for equatorial (0°N)
ocean surface or temperate (45°N) spring and fall ocean surface or
temperate (45°N) summer at deeper ocean depths; 8 h for temperate (45°N)
winter at the surface or at temperate (45°N) spring and fall at depth
and equatorial (0°N) deep ocean depths; and 4 h for temperate (45°N)
winter or deep ocean depths during temperate (45°N) spring and fall.

## Growth rate analysis {.unnumbered}
Data files (.csv) saved from the Multicultivator software were imported into R-Studio [@R-Studio] for data management [@R-tidyverse], growth rate calculations CITATION, comparisons of model fits (get Platt CITATION), and generation of
figures using `ggplot2` [@R-ggplot2]. The chlorophyll proxy
optical density (OD680 - OD720;  ΔOD) was used to determine the chlorophyll-specific exponential growth
rate (µ, d^-1^) for each treatment combination.  We first used a rolling mean from the RStudio `zoo` package [@R-zoo] to calculate the average ΔOD data over a 1-hour window to lower influence of outlier points and remove data points collected during post stationary phase, if applicable. We used the Levenberg-Marquardt algorithm
[@bellaviaLevenbergMarquardtMethod2018] modification of the non-linear
least squares,  using the R package `minpack.lm`
[@R-minpack.lm], to fit a logistic
equation \@ref(eq:GrowthRate)


```{=tex}
\begin{equation}
  µ = \frac{ΔOD_{max} × ΔOD_{min} × exp^{(µ × t)}}{ΔOD_{max} + (ΔOD_{min} × exp^{((µ × t) - 1)})}
  (\#eq:GrowthRate)
\end{equation}
```

where ΔOD~max~ is maximum ΔOD, ΔOD~min~ is minimum ΔOD, t is time
duration over the growth trajectory.  

Figure \@ref(fig:deltaODLogGrowthPlot) is an example of chlorophyll specific
growth estimates fitted from the high resolution ΔOD measurements for
each tube in a Multicultivator. The residuals of the logistic growth
curve fit are shown and the growth spectral waveband is plotted and
illustrates the imposed PAR (µmol photons m^-2^ s^-1^) and photoperiod
(h). Annotated code for data import, transformations and analyses is available at https://github.com/FundyPhytoPhys/prochlorococcus_o2/tree/main/Data/RawData


## Generalized additive model of growth responses {.unnumbered}

A Generalized Additive Model (GAM) CITATION was applied to the
relation between chlorophyll-specific µ, d^-1^  to photoperiod and PAR level,  for each growth [O~2~] level, and for the blue and red wavebands for growth, for each *P.marinus* strain in this study. The gam function from the R package `mgcv`
[@R-mgcv] was used to model the growth rate with smoothing terms, to
indicate the 90, 50 and 10% quantiles for growth rate across the levels of factors. Growth rate estimates with a standard error below 30% of the fitted growth rate were included in the GAM modelling. Our priority
was the effects of ecologically relevant blue light on growth trends.  We included analyses of growth responses to red light, which is not ecophysiologically relevant, but which might prove mechanistically informative [@murphyPhotoinactivationPhotosystemII2017].

## Estimation of photosynthetically usable radiation {.unnumbered}

To estimate the Photosynthetic Usable Radiation (PUR), a proxy of incident photons absorbed by the cells, for each *P. marinus* ecotype, the imposed Photosynthetic Active
Radiation (PAR) was first determined using the reported delivery of sinusoidal diel PAR regimes by the Multicultivators, point validated using an XXXXXcitation of PAR meterXXX. An emission profile from 400 nm to 700 nm of each coloured LED
light of the MCMIX-OD Multicultivator and the white LED light of the MC1000-OD Multicultivator was obtained using a Jaz spectrometer (Ocean Optics, Inc.,Dunedin, FL, USA) equipped with a fiber
optic cable, HH2 FiberOpticJmp (Part number A901073, Malvern Panalytical
Ltd, Malvern, UK). Each LED spectrum was then normalized to its emission maximum.
An *in-vivo* whole cell absorbance spectrum for each *P. marinus* strain under each spectral growth condition was obtained using the Olis 14 UV/VIS Clarity
Spectrophotometer (Olis Inc, Bogart, GA, USA) to scan across range of λ = 350 nm to 750 nm at 1 nm intervals. The path length of the internally reflective cavity of the Olis spectrophotometer was corrected to a 1 cm path length using the Javorfi correction method [@javorfiQuantitativeSpectrophotometryUsing2006] on PRO 99 media subtracted whole cell absorbance spectra. The blank-corrected whole cell absorbance spectra were normalized to the absorbance maximum of divinyl chlorophyll *a* (Chl a~2~), determined for each spectra, falling between 400 nm and 460 nm. 

An integrated weighting Equation \@ref(eq:WeightedPUR) [@morelAvailableUsableStored1978] was used to determine the weighted PUR spectrum P(λ),

\begin{equation}
  P(λ) = A(λ) × E(λ) 
  (\#eq:WeightedPUR)
\end{equation}

where A(λ) is the blank subtracted, Chl a~2~ peak normalized whole cell absorbance spectrum for each *P. marinus* ecotype, over 400 nm to 700 nm, A(λ); and E(λ) is the peak normalized emission spectrum of the imposed LED growth light, over 400 nm to 700 nm.

Actinic PUR levels (µmol photons m^-2^ s^-1^) were calculated from actinic PAR (µmol photons m^-2^ s^-1^) levels using the equation \@ref(eq:PUR) from [@morelAvailableUsableStored1978],

\begin{equation}
  PUR = \frac{\int_{400}^{700} P(λ)}{\int_{400}^{700} E(λ)} × PAR
  (\#eq:PUR)
\end{equation}


where P(λ) is the weighted PUR absorbance spectrum from equation \@ref(eq:WeightedPUR), E(λ) is the imposed growth light emission spectrum from equation \@ref(eq:WeightedPUR) and PAR is the imposed peak light level (µmol photons m^-2^ s^-1^).  Figure (\@ref(fig:PurParPlot)) shows the calculated absorbed peak PUR (µmol photons m^-2^ d^-1^) versus imposed actinic peak PAR (µmol photons m^-2^ s^-1^) for each strain and each spectral waveband (nm).

The applied photoperiods were delivered using the sine circadian light function of the PSI Multicultivator to simulate light exposure approximating sun rise through to sunset. The area under the sinusoidal curves is equivalent to the area of a triangular photoregime of equivalent photoperiod (Campbell, unpub), therefore the equation to determine the cumulative diel PUR (µmol photons m^-2^ d^-1^) is one half of the base (photoperiod) multiplied by the height (PUR) (Equation \@ref(eq:SumPURDay)),

\begin{equation}
  Cumulative diel PUR = \frac{PUR × 3600 × Photoperiod}{2}
  (\#eq:SumPURDay)
\end{equation}

where PUR is the actinic absorbed light calculated from equation \@ref(eq:PUR) (µmol photons m^-2^ s^-1^), 3600 is the time conversion from seconds to hour and photoperiod is the imposed photoperiod (h). 



Figures \@ref(fig:MED4OverlayPlots), \@ref(fig:SS120OverlayPlots) and
\@ref(fig:MIT9313OverlayPlots) provide visual representations of PUR, the black solid line and shaded area, in relation to the imposed PAR, the dotted line, under each imposed spectral wavebands for *P. marinus* MED4, SS120 and MIT9313, respectively. Figure (\@ref(fig:PurParPlot)) shows the relationship between calculated PUR versus imposed PAR for each *P. marinus* and each spectral waveband.

\newpage

```{r PurParPlot, fig.cap = PurParPlot_cap , fig.height = 20, fig.width = 10, echo = FALSE}

PurParPlot_cap <- glue("**Absorbed peak Photosynthetically Usable Radiation (PUR) (µmol photons m^-2^ s^-1^) vs. peak Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^).**",
                       "The correlation between PAR, plotted on the x-axis and PUR, plotted on the y-axis, are coloured for each growth spectral waveband; 450 nm (blue circles), 660 nm (red circles) and white LED (black circles).",
                       "The grey dashed line represents a hypothetical one to one correlation.",
                       "**A.** is *Prochlorococcus marinus*  MED4.",
                       "**B.** is *Prochlorococcus marinus*  SS120.",
                       "**C.** is *Prochlorococcus marinus*  MIT9313.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PurParPlot.png"))
```


\newpage

## Quantitative protein immunoblotting {.unnumbered}

Quantitative protein immunoblots were used to determine the concentrations of 5 target proteins; FtsH, PsbA, PetC, PsaC and RbcL in *P. marinus* MED4 and MIT9313 cultures grown in the PSI Multicultivator under imposed PAR light levels of 30 and 100 µmol photons m^-2^ s^-1^, spectral wavebands of 660 and 450 nm and [O~2~] of 2.5 and 250 µM. 40 mL of culture were transferred into a 50 mL falcon tube and 40 µL of 10 % Pluronic F68 acid was added to the cell suspension. The cell suspension was centrifuged for 30 minutes at 17664 g using a J-26XP centrifuge (Beckman Coulter, Brea, CA, USA) equipped with a J-10 rotor. The supernatant was removed and the resulting pellet was flash frozen in liquid nitrogen and stored at -80 °C for later use.

Protein samples were thawed and placed in 2 mL FastPrep®-24 bead lysing matrix D vials (MP Biomedicals, Irvine, CA, USA; SKU 116913050) with 200 uL of 1X protein extraction buffer (0.0281 mol L^−1^ TRIS base buffer, 0.0211 mol L^−1^ TRIS HCl buffer, 0.0735 mol L^−1^ lithium dodecyl sulfate, 1.08 mol L^−1^ glycerol, 0.5 mmol L^−1^ ethylenediaminetetraacetic acid, 0.1 mg·mL−1 PefaBloc SC (AEBSF) protease inhibitor) [@brownFluxCapacitiesAcclimation2008]  XXX note we used a low tris PEB recipe not sure where it came from (citation??) XXX. Cells were lysed for 3 cycles of 60 seconds at 6.5m s-1 with 60 second intervals on ice between cycles.  The lysing vials were centrifuged at 16100g using XXXXXX centrifuge ( reference here    ) for 5 minutes, the supernatant was transferred to a 600 uL microcentrifuge tube and centrifuged again at 16100g for 3 minutes. 40 µL of the supernatant were subsampled into labeled vials and stored at -80 °C until analysis. The bicinchoninic acid (BCA) assay was used for total protein determination using a calibration curve of bovine gamma globulin (BGG; Bio-Rad, Hercules, CA, USA) standard ranging from 0 to 1.0 mg mL^-1^ range.

All samples and appropriate standards were prepared as described in [@brownFluxCapacitiesAcclimation2008]. Standards were obtained from Agrisera (Vännäs, Sweden; FtsH AS11 1789S; PsbA AS01 016S; PetC AS08 330S; PsaC AS04 042S; RbcL AS01 017S).  Between 0.1 to 4.0 µg of total protein was loaded onto 15 well Invitrogen Bolt 4-12% acrylamide Bis-Tris gels (ThermoFisher Scientific, Waltham, MA, USA). All gels were run for 25 minutes at 200V using Invitrogen Bolt Mini Gel tanks and Invitrogen Bolt MES SDS running buffer (ThermoFisher Scientific, Waltham, MA, USA) and transferred for 60 minutes at 20V onto a PVDF membrane (Bio-Rad, Hercules, CA, USA; SKU 1620177). Blots were blocked overnight in 2% ECL blocking solution as described by @brownFluxCapacitiesAcclimation2008. The following morning, each blot was incubated in their appropriate antibody obtained from Agrisera (Vännäs, Sweden; FtsH  1:5000 dilution, AS11 1789; PsbA  1:10000 dilution, AS05 084; PetC 1:10000 dilution, AS08 330; PsaC 1:5000 dilution, AS10 939; RbcL 1:10000 dilution, AS03 037) and a 1:15000 dilution of the secondary antibody, goat anti-rabbit HRP conjugated (Bio-Rad, Hercules, CA, USA; SKU 1706515, lot 64230791).  Blots were incubated in Amersham ECL Select chemiluminescent substrate (Danaher, Washington, DC, USA) and imaged in a Bio-Rad VersaDoc imager (Bio-Rad, Hercules, CA, USA). Images were quantitated using the ImageJ2 software (version 1.53c).

## *Prochlorococcus* comparative genomics {.unnumbered}

Using the dataset Omar *et al*. [@omarAnnotationGenesEncoding2023], we filtered the dataset for Enzyme Commission Numbers (EC numbers), or Kegg Orthology Numbers (KO numbers) identified by BRENDA  [@changBRENDAELIXIRCore2021] as 'natural substrates' for O~2~ in *P. marinus* (MED4, MIT9313, SS120, and NATL2A) in order to generate a subset containing only those orthologs encoding enzymes directly mediating O~2~ metabolism. We grouped orthologs together by EC number and their KO number (KO number) and determined the occurrences of individual orthologs encoding each EC number, or KO number when EC number was not available, in a given strain. We merged the dataset with a list of enzyme Michaelis constant (K~m~) values from other organisms, as K~m~ values from *Prochlorococcus* were only available in the case of Ribulose bisphosphate carboxylase. XXX Possibly add the link to the full list of genes. XXXX


\newpage

# Results and discussion {.unnumbered}

## Detection of *Prochlorococcus* proteins across O~2~ and light niches in the ocean  {.unnumbered}

Proteins from 13 annotated strains of *P. marinus* were detected across depths and oxygen concentrations in the data set analyzed. We have focused our analysis here on photosynthetically relevant protein complexes from three strains: HLI MED4; LLIII SS120 and LLIV MIT9313. Figure \@ref(fig:NATLProchloroPhotosynthDepthO2) plots the observation of core photosynthetic complexes (Photosystem II, the Cytochromeb6f complex, Phototsystem I, ATP Synthase and Rubisco) in these strains as a function of depth (a proxy for light intensity) and measured [O~2~]. Photosynthetic complexes from MED4 were detected throughout the water column predominantly at high [O~2~]. Complexes from LLIII strain SS120 were detected throughout the water column in high [O~2~] and at depth in low [O~2~] samples. Photosystem I and II were only detected at low [O~2~]. The exclusive assignment of ATP Synthase peptides to MIT9313 may result from the high conservation of protein sequence for this complex.

XXXMore TextXXX from Amanda & AuroraXXX
Proteins derived from Clade HLXXX ecotypes of *P. marinus* were detected in OMZ at depths up to 200 meters,  with O~2~ of 15 µM. The extent to which [O~2~] defines the niches occupied by different *P. marinus* ecotypes, as compared to potentially covarying environmental variables like photoperiod, light spectrum, and light level, is poorly described.

\newpage

```{r NATLProchloroPhotosynthDepthO2, fig.cap = NATLProchloroPhotosynthDepthO2_cap , fig.height = 20, fig.width = 10, echo = FALSE}

NATLProchloroPhotosynthDepthO2_cap <- glue("**Ocean detection of *Prochlorococcus marinus* photosynthesis complexes.**", 
                       "Protein detections are plotted vs. O~2~ (µM) (X axis) and depth (m) (Y axis) at sample origin.",
                       "Rows separate data annotated as from *Prochlorococcus marinus* strains MED4 (Clade HLI), SS120 (Clade LLIII) and MIT9313 (Clade LLIV).",
                       "Columns show detections of proteins annotated as  Photosystem II (PSII), Cytochromeb6f complex (Cytb6f), Photosystem I (PSI), ATP Synthase or Ribulose-1,5-bisphosphate oxygenase carboxylase (RUBISCO).",
                       "For comparison culture growth experimental conditions are indicated by horizontal grey lines for depths approximating imposed peak Photosynthetically Active Radiation (µmol photons m^-2^ s^-1^); and vertical grey lines for [O~2~] (µM).",
                       "Point colours identify *Prochlorococcus* clade assignments",
                       "Data from OceanProteinPortal (https://www.oceanproteinportal.org/).",
 .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "NATLProchloroPhotosynthDepthO2.png"))
```


\newpage

## *Prochlorococcus marinus* growth responses to photoperiod, PAR, spectral band, and [O~2~]  {.unnumbered} 

Guided by the evidence of ocean distributions of proteins from *Prochlorococcus* we set up a matrix of photoperiods, PAR, spectral bands, and [O~2~] to approximate current, and potential future, latitudinal, depth and seasonal niches for *Prochlorococcus* strains.  We earlier found [@murphyPhotoinactivationPhotosystemII2017] that growth under red light could prove mechanistically informative as to factors limiting *Prochlorococcus* growth, so we included that spectral band even though it is not characteristic of representative *Prochlorococcus* niches.  Although *Prochlorococcus* is currently limited to a narrow range of surface photoperiods, potential poleward latitudinal expansions, in combination with attenuation of light with depth, mean *Prochlorococcus* currently, or potentially, encounters a wide range of photoperiods.  Our growth rate determinations generally agreed with those from @mooreComparativePhysiologySynechococcus1995, for white LED and 250 µM O~2~, but our study is, we believe, the first to analyze the interactive growth responses of *Prochlorococcus* strains to varying [O~2~], spectral wavebands and photoperiods. 

### MED4 Clade HLI {.unnumbered}

Under 250 µM O~2~ the growth rates of *P. marinus* MED4, Clade HLI, increased with higher imposed PAR and longer photoperiods
(Figure \@ref(fig:MED4PAR)), across all spectral wavebands. No growth was observed under any imposed conditions under a 4 h photoperiod. The maximum growth rate (µ~max~) was `r PURPARFits %>% filter(Strain == "MED4", WL == "450", O2_uM == "250", Photoperiod == 16) %>% select(ReplicateMean_deltaOD_Lmu_corr_day) %>% max() %>% round(digits = 2)` d^-1^ achieved under 180 µE blue light and 16 h photoperiod.

Similar to growth trends under 250 µM O~2~, MED4 maintained at 25 µM O~2~ showed fastest growth when the
photoperiod was 16 h for each spectral waveband, across PAR levels (Figure \@ref(fig:MED4PAR)).
The µ~max~ was `r PURPARFits %>% filter(Strain == "MED4", WL == "450", O2_uM == "25", Photoperiod == 16) %>% select(ReplicateMean_deltaOD_Lmu_corr_day) %>% max() %>% round(digits = 2)` d^-1^ (Table. \@ref(tab:MaxGrowthTable)) achieved under 180 µmol photons m^-2^ s^-1^ blue light and 16 h photoperiod. The 4 h photoperiod
experiments under white LED light were not performed because no growth was achieved when grown under an 8 h photoperiod of white LED light.

MED4 did not grow under when sparged to the lowest [O~2~] of 2.5 µM (Figure \@ref(fig:MED4PAR)). 2.5 µM O~2~ growth experiments
were not conducted for 4 and 16 h photoperiods, as no reproducible growth occurred when MED4 was exposed to 8 and 12 h photoperiods.

\newpage

```{r MED4PAR, fig.cap = MED4PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MED4PAR_cap <- glue("**Chlorophyll specific growth rate (d^-1^) for *Prochlorococcus marinus* MED4 (High-Light (HLI) near surface clade) vs. photoperiod (h).  **",
                    "Rows show levels of imposed dissolved O~2~ concentrations as 250 µM, 25 µM and 2.5 µM.",
                       "Columns show 3 levels of growth Photosynthetically Active Radiation (PAR); 30, 90 and 180 µmol photons m^-2^ s^-1^",
                    "colours represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MED4PAR.png"))
```

\newpage

The GAM model in Figure \@ref(fig:MED4GAM) summarizes MED4 growth responses to red (A,B) or blue (C,D) peak PAR and photoperiod across 2 imposed oxygen concentrations.  Under 250 µM O~2~ MED4 achieved fastest growth rates above peak blue light of ~180 µmol photons m^-2^ s^-1^, and the longest photoperiod of 16, indicated by the 0.64 d^-1^ contour line representing the 90^th^ percentile of maximum achieved growth rate (Figure \@ref(fig:MED4GAM)C).  Growth decreased with decreasing photoperiod and decreasing peak PAR. Under red light growth was generally slower but the pattern of growth responses to photoperiod and PAR was similar (Figure \@ref(fig:MED4GAM)A).  Note the exclusion of MED4 from growth under 4 h photoperiod under both red and blue light (Figure \@ref(fig:MED4GAM)). Under 25 µM O~2~ MED4 showed similar growth responses, but was excluded from both 4 and 8 h photoperiods.  MED4 did not grow under 2.5 µM O~2~, so no GAM model was run. 
Considering the range of PAR levels, and spectral bands that MED4 can utilize, MED4 can inhabit not just shallow depths, where light levels are high, but also deeper regions, characterized by a lower level of blue light, subject to the limitation of a photoperiod of more than 4 h, even after depth attenuation of light. The photoregimes of winter temperate zones, due to shorter photoperiods, exclude MED4 from growth at any depth, however temperate photoperiods and light levels for the remainder of the year are potentially adequate to support MED4 growth, if water temperatures warm into the Clade HLI tolerance range.

\newpage

```{r MED4GAM,  fig.cap = MED4GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

MED4GAM_cap <- glue("**A contour plot of a Generalized Additive Model (GAM) representing the chlorophyll specific growth rate (d^-1^) for *Prochlorococcus marinus* MED4 grown under 660 nm (red) or 450 nm (blue) light.**",
                    "X-axis is photoperiod (h).",
                    "Y-axis is actinic Photosynethetically Active Radiation (PAR, µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 250 µM of O~2~ and blue light.",
                    "**D.** represents the model under 25 µM of O~2~ and blue light.",
                    "Legends represent a colour gradient of growth rate from no growth (white) to 1.00 d^-1^ (dark red or dark blue).",
                    "Labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_MED4_Oxy_.png"))


```

\newpage

### SS120 Clade LLIII {.unnumbered}

Under 250 µM O~2~ growth rates of *P. marinus* SS120, Clade LLIII, increased with longer photoperiods, under 30 µmol photons m^-2^ s^-1^ peak PAR and (Figure \@ref(fig:SS120PAR)). No growth was observed under any blue light photoperiods when exposed to peak PAR of 90 µmol photons m^-2^ s^-1^ or greater. Growth rate, however increased with increasing photoperiods for white and red light under peak PAR of 90 µmol photons m^-2^ s^-1^.  Growth rate decreased with longer photoperiods and showed growth inhibition at 12 and 16 h photoperiods under PAR of 180 µmol photons m^-2^ s^-1^  white, red or blue LED light.  The µ~max~ was `r PURPARFits %>% filter(Strain == "SS120", WL == "WW", O2_uM == "250", Photoperiod == 16) %>% select(ReplicateMean_deltaOD_Lmu_corr_day) %>% max() %>% round(digits = 2)` d^-1^ (Table. \@ref(tab:MaxGrowthTable)) achieved under 90 µmol photons m^-2^ s^-1^ white LED light and 16 h photoperiod.

Under 25 µM O~2~ and PAR of 30 µmol photons m^-2^ s^-1^ growth trends were similar to 250 µM O~2~. SS120 showed no
growth under a 4 h photoperiod for red spectral waveband, however under blue light, SS120 was able to grow under a 4 h photoperiod (Figure
\@ref(fig:SS120PAR)). In contrast to the growth trends of the 250 µM O~2~ and PAR of 90 µmol photons m^-2^ s^-1^ experiments, SS120 grew under 4 and 8 h blue light and 16 h red light photoperiods, however the growth rate decreased under 12 and 16 h white LED light photoperiod treatments. Blue light treatments under PAR of 180 µmol photons m^-2^ s^-1^ showed growth only under an 8 h photoperiod. The µ~max~ was `r PURPARFits %>% filter(Strain == "SS120", WL == "450", O2_uM == "25", Photoperiod == 8) %>% select(ReplicateMean_deltaOD_Lmu_corr_day) %>% max() %>% round(digits = 2)` d^-1^ (Table. \@ref(tab:MaxGrowthTable)) achieved under 90 µmol photons m^-2^ s^-1^ blue light and 8 h photoperiod. The 25 µM O~2~, less than 16 h photoperiod and 180 µmol photons m^-2^ s^-1^  under white LED light experiments were not performed due to time constraints.

SS120 did not reproducibly grow when sparged to the lowest O~2~ of 2.5 µM (Figure \@ref(fig:SS120PAR)). 2.5 µM O~2~ growth experiments were not conducted for 4 and 16 h photoperiods under PAR of 180 µmol photons m^-2^ s^-1^, as no growth occurred when SS120 was exposed to 8 and 12 h photoperiods. Red light 16 h photoperiod experiments were not performed due to time constraints.
\newpage

```{r SS120PAR, fig.cap = SS120PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

SS120PAR_cap <- glue("**Chlorophyll specific growth rate (d^-1^) for *Prochlorococcus marinus* SS120 (Low-Light (LLIII) deep ocean clade) vs. photoperiod (h). **",
                     "Rows show levels of imposed dissolved O~2~ concentrations as 250 µM, 25 µM and 2.5 µM.",
                       "Columns show 3 levels of growth Photosynthetically Active Radiation (PAR); 30, 90 and 180 µmol photons m^-2^ s^-1^",
                    "colours represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",
                    
                       .sep = " ")
                       
knitr::include_graphics(file.path(FiguresIn, "SS120PAR.png"))

```

\newpage

The GAM model in Figure \@ref(fig:SS120GAM) summarizes growth responses of SS120 to red (A,B) or blue (C,D) peak PAR and photoperiod, across the 2 imposed oxygen concentrations. Under 250 µM O~2~, Figure \@ref(fig:SS120GAM)C showed highest growth rates below PAR of 50 µmol photons m^-2^ s^-1^ and photoperiods between 8 and 12 h, indicated by the contour line labeled 0.19 d^-1^ (representing the 90^th^ percentile of achieved growth rate).  Under 250 µM O~2~ SS120 is constrained to deeper ocean waters through its intolerance of higher blue PAR levels. These findings align with @mooreComparativePhysiologySynechococcus1995 and are expected for a low light clade.  The disjunct regions of the GAM plot results from variable growth success of SS120 under 250 µM O~2~.  Growth rate patterns under red light were similar, although somewhat faster. In contrast, under 25 µM O~2~ and a photoperiod of 8 h SS120 exploited all blue peak PAR levels, achieving faster growth rates at a higher PAR of \~100 µmol photons m^-2^ s^-1^, indicated by the contour line labeled 0.4 d^-1^ (representing the 90^th^ percentile of achieved growth rate), out pacing the 90^th^ percentile fastest growth rates under 250 µM O~2~. Under red light and 25 µM O~2~ (Figure \@ref(fig:SS120GAM)B) SS120 grew across most conditions of peak PAR and photoperiod, achieving fastest growth under long photoperiods and peak PAR below 50 \~100 µmol photons m^-2^ s^-1^.  Thus, the designation of SS120 as a LL strain is dependent upon the [O~2~]. SS120 did not, however, grow reliably under tested conditions at 2.5 µM O~2~.

\newpage

```{r SS120GAM,  fig.cap = SS120GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

SS120GAM_cap <- glue("**Contour plot of a Generalized Additive Model (GAM) representing the chlorophyll specific growth rate (d^-1^) for *Prochlorococcus marinus* SS120 grown under 660 nm (red) or 450 nm (blue) light.**",
                    "X-axis is photoperiod (h).",
                    "Y-axis is actinic Photosynethetically Active Radiation (PAR, µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 250 µM of O~2~ and blue light.",
                    "**D.** represents the model under 25 µM of O~2~ and blue light.",
                    "Legends represent a colour gradient of growth rate from no growth (white) to 1.00 d^-1^ (dark red or dark blue).",
                    "Labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_SS120_Oxy_.png"))


```

\newpage

### MIT9313 Clade LLIV {.unnumbered}

Under 250 µM O~2~ growth rates of *P. marinus* MIT9313, Clade LLIV, increased with longer
photoperiods, under low 30 µmol photons m^-2^ s^-1^ peak PAR, (Figure \@ref(fig:MIT9313PAR)).  Under intermediate 90 µmol photons m^-2^ s^-1^ peak PAR growth rates decreased with increasing blue light photoperiod. Blue light did not induce growth at 180 µmol photons m^-2^ s^-1^ peak PAR, while MIT9313 showed only marginal growth under white LED and red light at 180 µmol photons m^-2^ s^-1^ peak PAR, under the 8 h photoperiod, consistent with [@moorePhotophysiologyMarineCyanobacterium1999]. The µ~max~ was `r PURPARFits %>% filter(Strain == "MIT9313", WL == "450", O2_uM == "250", Photoperiod == 16) %>% select(ReplicateMean_deltaOD_Lmu_corr_day) %>% max() %>% round(digits = 2)` d^-1^ achieved under 30 µmol photons m^-2^ s^-1^ blue light and 16 h photoperiod.

For MIT9313 under 25 µM O~2~ growth rate increased with increasing photoperiods for all spectral wavebands tested (Figure \@ref(fig:MIT9313PAR)), with the fastest overall growth rate for MIT9313 
`r PURPARFits %>% filter(Strain == "MIT9313", WL == "WW", O2_uM == "25", Photoperiod == 16, Par_ue == 90) %>% select(ReplicateMean_deltaOD_Lmu_corr_day) %>% max() %>% round(digits = 2)` d^-1^ achieved under peak PAR of 90 µmol photons m^-2^ s^-1^ and 16 h white LED light photoperiod. In marked contrast to the 250 µM O~2~ growth experiments, MIT9313 grew when exposed to peak PAR of 180 µmol photons m^-2^ s^-1^ and blue light under all photoperiods except 16 h; additionally, white LED and red light treatments induced growth across all tested photoperiods under 25 µM O~2~. The 25 µM O~2~, 4 h photoperiod experiments under white LED light and were not performed due to time constraints.

MIT9313 grew under 2.5 µM O~2~  particularly under blue LED light, albeit generally slower than under the parallel experiments at 25 µM O~2~  (Figure \@ref(fig:MIT9313PAR)). Growth estimates showed scatter among replicates, suggesting  2.5 µM O~2~  is near the tolerance limit for growth of MIT9313. Growth rates increased with longer photoperiods under blue light treatments under peak PAR of 90 µmol photons m^-2^ s^-1^ . Growth  for MIT9313 under PAR of 180 µmol photons m^-2^ s^-1^ and blue light treatment decreased with increasing photoperiods with full growth inhibition under a 16 h photoperiod. The red light peak PAR of 180 µmol photons m^-2^ s^-1^ showed similar growth rates for 8 and 12 h photoperiods. The µ~max~ was `r PURPARFits %>% filter(Strain == "MIT9313", WL == "450", O2_uM == "2.5", Photoperiod == 12, Par_ue == 90) %>% select(ReplicateMean_deltaOD_Lmu_corr_day) %>% max() %>% round(digits = 2)` d^-1^ achieved under 12 h blue light photoperiod and PAR of 90 µmol photons m^-2^ s^-1^. The 2.5 µM O~2~ white LED treatments under 4, 8 and 16 h photoperiods and red light under 4 and 16 h photoperiods were not performed due to time constraints.

\newpage

```{r MIT9313PAR, fig.cap = MIT9313PAR_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MIT9313PAR_cap <- glue("**Chlorophyll specific growth rate (d^-1^) for *Prochlorococcus marinus* MIT9313 (Low-Light (LLIV) deep ocean clade) vs. photoperiod (h). **",
                       "Rows show levels of imposed dissolved O~2~ concentrations as 250 µM, 25 µM and 2.5 µM.",
                       "Columns show 3 levels of growth Photosynthetically Active Radiation (PAR); 30, 90 and 180 µmol photons m^-2^ s^-1^",
                    "colours represent the actinic spectral waveband (nm).",
                    "Large circles show mean or single determinations of growth rate from logistic curve fits; small circles show values for replicate determinations, if any: replicates often fall with larger circles.",

                       .sep = " ")
                       
knitr::include_graphics(file.path(FiguresIn, "MIT9313PAR.png"))

```

\newpage

The GAM model in Figure \@ref(fig:MIT9313GAM) summarizes MIT9313 growth responses to red (A,B,C) or blue (D,E,F) peak PAR and photoperiod. Under 250 µM O~2~, Figure \@ref(fig:MIT9313GAM)D shows MIT9313 achieves fastest growth rates between blue peak PAR of 30 µmol photons m^-2^ s^-1^ and 50 µmol photons m^-2^ s^-1^ and photoperiods longer than 8 h, indicated by the contour line labeled 0.52 d^-1^ representing the 90^th^ percentile of achieved growth rates. Figure \@ref(fig:MIT9313GAM)D also shows that growth rate increases with longer photoperiods, as long as the blue peak PAR levels remain below 50 µmol photons m^-2^ s^-1^.  In contrast, under red light and 250 µM O~2~ MIT9313 grows faster while exploiting higher peak PAR and longer photoperiods.  Figure \@ref(fig:MIT9313GAM)F shows that MIT9313 maintains growth even under 2.5 µM O~2~, under photoperiods between 4 and 8 h and peak blue PAR between 50 to 10 µmol photons m^-2^ s^-1^ PAR.  Thus the designation of MIT9313 as a LL clade is dependent upon [O~2~] and light spectra. (Figure \@ref(fig:MIT9313GAM)E).

\newpage

```{r MIT9313GAM,  fig.cap = MIT9313GAM_cap, fig.height = 10, fig.width = 10, echo = FALSE}

MIT9313GAM_cap <- glue("**Contour plot of a Generalized Additive Model (GAM) representing the chlorophyll specific growth rate (d^-1^) for *Prochlorococcus marinus* MIT9313 grown under 660 nm (red) or 450 nm (blue) light.**",
                    "X-axis is photoperiod (h).",
                    "Y-axis is actinic Photosynthetically Active Radiation (PAR, µmol photons m^-2^ s^-1^).",
                    "**A.** represents the model under 250 µM of O~2~ and red light.",
                    "**B.** represents the model under 25 µM of O~2~ and red light.",
                    "**C.** represents the model under 2.5 µM of O~2~ and red light.",
                    "**D.** represents the model under 250 µM of O~2~ and blue light.",
                    "**E.** represents the model under 25 µM of O~2~ and blue light.",
                    "**F.** represents the model under 2.5 µM of O~2~ and blue light.",
                    "Legends represent a colour gradient of growth rate from no growth (white) to 1.00 d^-1^ (dark red or dark blue).",
                    "Labeled contour lines indicate the 90%, 50%, and 10% quantiles for achieved growth rate.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "PICO_Hypervolume_GAM_MIT9313_Oxy_.png"))


```


\newpage

## PUR and growth responses {.unnumbered}

Cumulative diel PUR can potentially collapse photoperiod, PAR and spectral wavebands to a common metric of photosynthetically active light absorbed per day. Cumulative diel PUR dose (µmol photons m^-2^ d^-1^) was calculated from the imposed PUR (µmol photons m^-2^ s^-1^) and photoperiod (h). We plotted growth rates vs. cumulative diel PUR to determine whether growth is a simple response to diel PUR, across imposed photoperiods and spectral wavebands, or whether spectral wavebands or photoperiods have specific or interactive influences on growth beyond cumulative diel PUR.
  
Due to the absorption of *P. marinus* pigments in the blue spectral waveband range, the maximum imposed cumulative diel PUR under blue light is almost 3 times that of white LED light, and about 6 times the red light treatment, despite being derived from the same imposed photoperiods and peak PAR regimes. As such, only blue light experiments extend beyond cumulative diel PUR of ~ 2 x 10^6^ µmol photons m^-2^ d^-1^. This bias in the range of data leads to caution in comparing model fits of growth in response cumulative diel PUR under red, vs. blue, wavebands.

### MED4 Clade HLI {.unnumbered}

The representative of HLI clade, *P. marinus* MED4, showed no growth under any 4 h photoperiod treatments, even when a 4 h photoperiod delivered cumulative diel PUR equivalent to other photoperiod treatments. In parallel MED4 showed no growth under 2.5 µM O~2~, no matter the level of diel cumulative PUR. 
In contrast, under 250 or 25  µM O~2~, and under any photoperiod greater than 4 h, MED4 growth under blue light was well described by a saturating response (REFERENCE for EQUATION) of growth to increasing cumulative diel blue PUR, with saturation of growth rate achieved around  1.0 x 10^6^ µmol m^-2^d^-1^ (Figure \@ref(fig:BluevsRedPurFitsPlots)A), and no evidence of inhibition of growth at any achieved cumulative diel PUR.  Under the 'artificial' growth treatment of red light, MED4 achieved more growth per unit diel cumulative PUR  \@ref(fig:BluevsRedPurFitsPlots)A), consistent with [@murphyPhotoinactivationPhotosystemII2017], who showed a lower cost for growth under red light, for MED4, because red light provokes less photoinactivation of PSII, than equivalent levels of blue light. For distinct photoperiod fits refer to figure \@ref(fig:AllStrainPurFitsPlots)A.

### SS120 Clade LLIII {.unnumbered}

The representative of the LLIII clade, *P. marinus* SS120 showed almost no growth under 2.5 µM O~2~ experiments. Most 4 h photoperiod treatments of SS120 also did not grow under 250 µM O~2~, even when a 4 h photoperiod delivered cumulative diel PUR equivalent to other photoperiod treatments. SS120 did not grow when exposed to more than ~1.0 x 10^6^ µmol photons m^-2^ d^-1^ of cumulative diel PUR under any spectral waveband or photoperiod combination, under 250 µM O~2~ experiments(Figure \@ref(fig:BluevsRedPurFitsPlots)B).

Under both 25 or 250 µM O~2~ experiments, SS120 growth plateaued by about 5.0 x 10^5^ µmol photons m^-2^ d^-1^ diel PUR, with some scatter among photoperiod and spectral waveband regimes. The onset of growth inhibition extended to higher cumulative diel PUR for cultures under 25 µM O~2~, showing that SS120 is partially protected from photoinhibition of growth by 25 µM O~2~.  Under 25 µM O~2~ red light again generated more growth of SS120 per unit cumulative diel PUR, than did blue light, again consistent with lower cost of growth through lower photoinactivation under red light.
For distinct photoperiod fits refer to figure \@ref(fig:AllStrainPurFitsPlots)B.


### MIT9313 Clade LLIV {.unnumbered}

The LLIV clade representative, *P. marinus* MIT9313, showed growth rising to a plateau by about 5 x 10^5^ µmol photons m^-2^ d^-1^ of cumulative diel PUR, with higher growth rates over a narrower plateau under 25 and 250 µM O~2~, compared to a wider, lower, flatter response to cumulative diel PUR under 2.5 µM O~2~. Above about 1.0 x 10^6^ µmol photons m^-2^ d^-1^ of cumulative PUR under 250 µM O~2~, MIT9313 showed full inhibition of growth, across photoperiods, and spectral wavebands.  In contrast, under 25 µM O~2~, MIT9313 showed a greatly extended exploitation of higher cumulative diel PUR, with full growth inhibition only above about 3.5 x 10^6^ µmol photons m^-2^ d^-1^.  Similarly, under 2.5 µM O~2~, MIT9313 grew more slowly, but only showed full growth inhibition above about 3.5 x 10^6^ µmol photons m^-2^ d^-1^ cumulative diel PUR (Figure \@ref(fig:BluevsRedPurFitsPlots)C).  

Our data support enhanced growth under conditions of low cumulative dielPUR and 660 nm (red) spectral bandwidth, as supported by [@murphyPhotoinactivationPhotosystemII2017] who found a lower cost of growth, due to decreased photoinactivation of PSII under red, compared to blue wavebands.  Interestingly, this protective effect of red light disappears for MIT9313 growing under 2.5 µM O~2~, possibly because photoinactivation is strongly suppressed under this low [O~2~].
For distinct photoperiod fits refer to figure \@ref(fig:AllStrainPurFitsPlots)C.


\newpage

```{r BluevsRedPurFitsPlots, fig.cap = BluevsRedPurFitsPlots_cap , fig.height = 10, fig.width = 10, echo = FALSE}

 BluevsRedPurFitsPlots_cap <- glue("**Chlorophyll specific growth rate (d^-1^) vs. cumulative diel Photosynthetic Usable Radiation (PUR, µmol photons m^-2^ d^-1^).**",
                    "Rows show levels of imposed dissolved O~2~ concentrations as 250 µM, 25 µM and 2.5 µM.",
                    "Columns are strains; MED4, SS120 and MIT9313.",
                    "Shapes show the imposed photoperiod (h); 4 h (solid square),  8 h (solid diamond), 12 h (solid circle), 16 h (solid upright triangle).",
                      "Symbol colours show the spectral waveband for growth; 660 nm (red symbols), and 450 nm (blue symbols).",
                      "Large symbols show mean of growth rate from logistic curve fits; small symbols show values for replicate determinations, if any.",
                    "Platt (add Platt citation here) 3 parameter model fit to 660 nm (red lines) and 450 nm (blue lines) growth data for each combination of strain and dissolved oxygen shown with solid lines (significantly different fits, pvalue < 0.05) or dashed lines (not significantly different fits, pvalue > 0.05).",
                       .sep = " ")


knitr::include_graphics(file.path(FiguresIn, "BluevsRedPurFitsPlots.png"))

```

\newpage

## Photosystem II maintenance, oxygen metabolism, and DNA repair as limitations on *Prochlorococcus* growth {.unnumbered}  

*Prochlorococcus* remain challenging to culture, as their reduced genomes – the smallest of any known oxyphototroph – render them partially dependent upon mutualistic heterotrophic bacteria to  detoxify reactive oxygen species [@morrisFacilitationRobustGrowth2008; @morrisDependenceCyanobacteriumProchlorococcus2011]. MED4, SS120 and MIT9313 have been successfully cultured in laboratories  [@mooreComparativePhysiologySynechococcus1995; @mooreCulturingMarineCyanobacterium2007], and used to show that ecotypic classifications correspond to biochemical differences among strains [@bonisteelStrainSpecificDifferences2018]. CITATONS OTHERS.

Under full atmospheric [O~2~] and bluelight, LL clades of *Prochlorococcus* are restricted to growth under low light, in part because they suffer photoinhibition of Photosystem II (PSII) through several paths, including direct absorbance of UV or blue light, in parallel with generation of Reactive Oxygen Species (ROS) if the electron flow is slowed, [@aroPhotoinhibitionPhotosystemII1993], producing damaging singlet oxygen (^1^O~2~) [@aroPhotoinhibitionPhotosystemII1993; @soitamoPhotoinhibitionMarinePicocyanobacteria2017; @murphyPhotoinactivationPhotosystemII2017; @hakalaEvidenceRoleOxygenevolving2005]. Repair of photoinactivated PSII relies on the removal of damaged PsbA [@mannInvolvementFtsHHomologue2000; @adamFtsHProteasesChloroplasts2005], followed by reassembly with newly synthesized PsbA [@nixonRecentAdvancesUnderstanding2010]. Degradation of PsbA is a rate-limiting step in recovery from photoinhibition [@kanervoD1ProteinDegradation1993], mediated largely by a heterohexamer of (FtsH12)3, a membrane-bound (Sacharz et al., 2015)(Zak et al., 2001) metalloprotease  (Chiba et al., 2002) (Yoshioka-Nishimura and Yamamoto, 2014).  [@adamFtsHProteasesChloroplasts2005; @boehmSubunitOrganizationSynechocystis2012a; @nixonRecentAdvancesUnderstanding2010; @komendaExposedNTerminalTail2007] (Pisareva et al., 2007; Sacharz et al., 2015),  @mannInvolvementFtsHHomologue2000, (AMANDA REF from ALGATECH).

*Prochlorococcus* genomes encode 4 FtsH proteins, henceforth referred to as FtsH1-4, homologs to the characterized FtsH isoforms of the model freshwater cyanobacterium *Synechocystis* FtsH, and with presumably parallel functions (table \@ref(tab:FtsHTable)). Upon a shift to higher light HLIII MED4 upregulated expression of FtsH1 and FtsH2 [@bonisteelStrainSpecificDifferences2018], homologs to the *Synechocystis* slr0228 and slr1604, implicated in PSII repair. In contrast representative LLIV strain MIT9313 showed less overall expression of the FtsH proteases, and thus has fewer FtsH serving each photosystem.  Furthermore, MIT9313 expressed primarily FtsH3, homologous to *Synechocystis* slr1463 involved in PSI biogenesis, and  FtsH expression did not increase in response to light stress in MIT9313. Through adaptation to steady low light, Clade LLIV *Prochlorococcus* instead allocate resources to processes other than dynamic regulation of PSII repair. 

\newpage

```{r BuildFtsHtable, echo = FALSE}

FtsHTable <- tibble(Organism = c("*Prochlorococcus marinus*", "*Synechocystis sp. PCC6803*", "Function"),
                    FtsH1 = c("FtsH1", "SlrO22", "PSII Repair"),
                    FtsH2 = c("FtsH2", "Slr1604", "PSII Repair"),
                    FtsH3 = c("FtsH3", "Slr1463", "PSI biogenesis"),
                    FtsH4 = c("FtsH4", "Slr1390", "Cell viability")
                    )

```

```{r FtsHTable, fig.height = 6, fig.width = 10, include = TRUE, echo = FALSE}

knitr::kable(FtsHTable,
             caption = "FtsH protease homologs in *Prochlorococcus marinus* and the model cyanobacterium *Synechocystis* sp. PCC6803. Protein homology determined by multiple sequence alignment with MUSCLE followed by construction of maximum likelihood phylogenetic tree using 1000 bootstrap replicates in MEGAX.",
              col.names =  c("Organism",
                          "Homolog 1",
                          "Homolog 2",
                          "Homolog 3",
                          "Homolog 4"),
               align = "lcccc"
             )
```

\newpage

Ocean detections of proteins mediating protein metabolism support this interpretation of distinct FtsH function across clades of *P. marinus*. Ribosome proteins from Clade HLI MED4, Clade LLIII SS120 and Clade LLIV MIT9313 show generally similar patterns vs. [O~2~] and depth, a proxy for peak PAR.  FtsH3, inferred to mediate PSI assembly, likewise shows a similar pattern between MED4 and MIT9313.  But only MED4 shows detected presence of the FtsH1 & FtsH2 isoforms inferred to mediate PSII repair, and then only in near-surface samples subject to higher light.  Furthermore, even though MIT9313 grows (Figure \@ref(fig:MIT9313GAM)), and is detected in the ocean at low [O~2~] (Figure \@ref(fig:ProchloroPhotosynthDepthO2)), no  FtsH from MIT9313 is detected at low [O~2~], showing limited requirement for protein turnover under low [O~2~].

\newpage

```{r NATLProchloroProteinMetabDepthO2, fig.cap = NATLProchloroProteinMetabDepthO2_cap , fig.height = 20, fig.width = 10, echo = FALSE}

NATLProchloroProteinMetabDepthO2_cap <- glue("**Ocean detection of *Prochlorococcus marinus* protein metabolism complexes.**", 
                       "Protein detections are plotted vs. O~2~ (µM) (X axis) and depth (m) (Y axis) at sample origin.",
                       "Rows separate data annotated as from *Prochlorococcus marinus* strains MED4 (Clade HLI), SS120 (Clade LLIII) and MIT9313 (Clade LLIV).",
                       "Columns show detections of proteins annotated as FtsH Protease Complexes (FtsH1, FtsH2, FtsH3) or the Ribosome).",
                       "Culture growth experimental conditions indicated by horizontal grey lines for depths approximating Photosynthetically Active Radiation (µmol photons m^-2^ s^-1^) and vertical grey lines for [O~2~] (µM).",
                       "Point colours identify *Prochlorococcus* clade assignments",
                       "Data from OceanProteinPortal (https://www.oceanproteinportal.org/).",
 .sep = " ")



knitr::include_graphics(file.path(FiguresIn, "NATLProchloroProteinMetabDepthO2.png"))
```

\newpage

XXXX Mireille to finish Fig. caption when figure is complete. Add colour ? Remove []? XXX

```{r  MED4ProteinAnnotated, fig.cap = MED4ProteinAnnotated_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MED4ProteinAnnotated_cap <- glue("**Bar graph indicating protein concentrations determined from lab grown *Prochlorococcus marinus* MED4 (High-Light (HLI) near surface clade.  **",
                                    "Growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) and spectral wavelength are in rows;  2 levels of imposed growth dissolved O~2~ concentrations (µM) are in columns.",
                                    "Numbers over each bar are fmole target protein per ug total protein.",
                                    .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MED4ProteinAnnotated.png"))

```

\newpage


XXXX Mireille to finish Fig. caption when figure is complete. Add colour ? Remove []? XXX

```{r  MIT9313ProteinAnnotated, fig.cap = MIT9313ProteinAnnotated_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MIT9313ProteinAnnotated_cap <- glue("**Bar graph indicating protein concentrations determined from lab grown *Prochlorococcus marinus* MIT9313 (Low-Light (LLIV) deep ocean clade).  **",
                                    "Growth Photosynthetically Active Radiation (PAR) (µmol photons m^-2^ s^-1^) and spectral wavelength are in rows;  2 levels of imposed growth dissolved O~2~ concentrations (µM) are in columns.",
                                    "Numbers over each bar are fmole target protein per ug total protein.",
                                    .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MIT9313ProteinAnnotated.png"))

```


\newpage

Figure \@ref(fig:ProchlorococcusEnzymeKms) shows genes encoding enzymes for  *P. marinus* strains from Clades HLI, LLII, LLIII and LLII [@omarAnnotationGenesEncoding2023], compared to the measured or inferred K~M~ for [O~2~] for each enzyme CITATIONS. MED4 increases expression of alternative oxidase ('ubiquinol oxidase (non electrogenic)')  to cope with changes in light [@bergResponsesPsbAHli2011], by dissipating electrons from the inter-system transport chain. The approximate K~M~ for [O~2~] of ~ 25 µM for ubiquinol oxidase (non electrogenic) is comparable to the lower limit for growth of MED4 in our experiments.  We suggest that dependence upon this enzyme excludes MED4 from low oxygen zones.  The genome scan shows SS120 and MIT9313 lack this gene, and therefore, lack this oxygen-dependent path to cope with changing excitation.  Conversely, a gene encoding (S)-2-hydroxy-acid oxidase is encoded in the MIT9313 genome.  (S)-2-hydroxy-acid oxidase catalyzes the reaction of 2-hydroxy acid with O~2~ to produce toxic H~2~O~2~ [@InformationEC15]. (S)-2-hydroxy-acid oxidase has an approximate K~M~ for [O~2~] of ~ 250 µM, and produces H~2~O~2~, so growth at lower [O~2~] may protect MIT9313 from auto-intoxication from production of H~2~O~2~. We hypothesize that under 250 µM O~2~ and higher blue light, *P. marinus* MIT9313 suffered photoinhibition, resulting from the inactivation of PSII caused by the production of the reactive oxygen species, hydrogen peroxide. This photoinhibition is compounded by the limited inducible repair mechanism for PSII, due to the absence of FtsH 2 and 3  expression in *P. marinus* MIT9313 [@bonisteelStrainSpecificDifferences2018].  We hypothesize that under the conditions of our high light and 2.5 µM and 25 µM O~2~ experiments, the activity of the (S)-2-hydroxy-acid oxidase enzyme is suppressed. As a result, the catalyzed production of hydrogen peroxide is inhibited, leading to less PSII damage, allowing MIT9313 to avoid photoinhibition and circumvent its limitations on PSII repair.  Figure \@ref(fig:ProchlorococcusEnzymeKms) also shows that *P. marinus* SS120 is the only tested ecotype to lack the pyridoxal 5'-phosphate synthase enzyme. The pyridoxal 5'-phosphate synthase enzyme is an important cofactor in the biosynthesis of vitamin B~6~ [@sangIdentificationPyridoxinePyridoxamine2007]. Vitamin B~6~ is a potential antioxidant and can effectively quench singlet oxygen [@bilskiVitaminB6Pyridoxine2000]. The absence of the pyridoxal 5'-phosphate synthase enzyme may explain why *P. marinus* SS120 does not grow as well as *P. marinus* MIT9313, when exposed to high light stress under 25 µM O~2~ and not at all under 2.5 µM O~2~.

\newpage

```{r ProchlorococcusEnzymeKms, fig.cap = ProchlorococcusEnzymeKms_cap, fig.height = 15, fig.width = 15, echo = FALSE}

ProchlorococcusEnzymeKms_cap <- glue("**K~m~ values for oxygen metabolizing enzymes.**" ,
                                     "The y-axis represents the log10 concentration of oxygen substrate (mM).",
                                     "The x-axis represents the oxygen metabolizing enzymes encoded in at least one of the *Prochlorococcus marinus* strains in this study.",
                                     "The *Prochlorococcus marinus* strains are indicated in rows." ,
                                     "The solid circles represent K~m~ values from literature and the asterisks represent predicted values.",
                                     "colours represent the gene counts.",
                                     "The red shaded area denotes a K~m~ oxygen concentration range from 230 to 280 µM.",
                                     "The green shaded area denotes a K~m~ oxygen concentration range from 5 to 50 µM . ",
                                     "The blue shaded area denotes a K~m~ oxygen concentration range from 0.5 to 5 µM.  ",
                                     "The black bars show the minimum and maximum K~m~ values.",
                       "Figure was generated using a filtered subset of the annotated phytoplankton gene sequences dataset from @omarAnnotationGenesEncoding2023.",
                       
                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "ProchlorococcusEnzymeKms.png"))

```

\newpage

Figure \@ref(fig:DNARepairFig) shows genes encoding DNA repair for *P. marinus* strains. As expected, *P. marinus* MED4 possesses the largest, most complete suite of genes encoding DNA repair enzymes, followed by *P. marinus* MIT9313. Conversely, *P. marinus* SS120 demonstrates the smallest genomic capacity for DNA repair. *Prochlorococcus marinus* MED4 was the only strain to possess a gene encoding deoxyribodipyrimidine photolyase, which, in the presence of blue light, is responsible for repairing DNA damaged by UV light [@sancarStructureFunctionDNA2003]. *Prochlorococcus marinus* MED4 was also the only strain to possesses a gene encoding DNA ligase, which uses ATP as a cofactor for DNA repair. The absence of  genes encoding deoxyribodipyrimidine photolyase and DNA ligase (ATP) in *P. marinus* MIT9313 and *P. marinus* SS120 helps explain why these two strains cannot tolerate growth under full [O~2~] and high light, found at the ocean surface. Furthermore, the protective effect of lower [O~2~], allowing these strains to grow at higher light, may relate in part to suppression of DNA damage when generation of Reactive Oxygen Species is suppressed at lower [O~2~].  *Prochlorococcus* are highly susceptible to hydrogen peroxide (H~2~O~2~) toxicity as they lack genes which scavenge H~2~O~2~ molecules [@morrisFacilitationRobustGrowth2008]. The small cell size of *Prochlorococcus* allow the reactive oxygen species (ROS), H~2~O~2~, to cross the cell membrane [@omarGenomicCapacitiesReactive2023]; however, accumulation of extracellular H~2~O~2~ remains toxic to *Prochlorococcus* [@morrisFacilitationRobustGrowth2008,  @morrisDependenceCyanobacteriumProchlorococcus2011].

\newpage

```{r DNARepairFig, fig.cap = DNARepairFig_cap, fig.height = 10, fig.width = 10, echo = FALSE}

DNARepairFig_cap <- glue("**Genes encoding DNA repair enzymes.**" ,
                         "The y-axis represents  *Prochlorococcus marinus* strains.",
                         "The x-axis represents enzymes encoded for DNA repair found in at least one *Prochlorococcus marinus* strain in this study.",
                         "The red solid circles represent a gene count of 1.",
                         "The green solid circles represent a gene count of 2.",
                         "The blue solid circles represent a gene count of 3.",
                       "Figure was generated using a filtered subset of the annotated phytoplankton gene sequences dataset from @omarAnnotationGenesEncoding2023.",
                       
                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "DNARepairFig.png"))

```


\newpage

The potential for niche expansion into temperate regions by *P. marinus* varies depending on the season, which influences achieved underwater photoperiods and light levels. Temperate summer delivers 11 hours of blue waveband light
underwater, above the photic threshold of 20 µmol photons m^-2^ s^-1^, while temperate spring/fall delivers 8 hours of blue waveband light
underwater, photoperiod ranges which are permissive for growth of all three *P. marinus*. In contrast temperate winter delivers only about 2 h
of blue waveband light underwater above the photic threshold, which precludes growth of MED4 and SS120, even if winter waters reached
permissive temperatures. MIT9313 and SS120 will be excluded from near-surface growth niches by high PAR, unless OMZ zones extend to the near surface.

Diverse *P. marinus* strains  @billerProchlorococcusStructureFunction2015 differentially exploit potential photoregimes, both at the surface and deep in the water column. Some *P. marinus* strains grow under low oxygen environments, similar to OMZ. The LL clades we tested can function as 'HL'  in oxygen environments of 25 µM, and as low as 2.5 µM, in the case of MIT9313.

@westNichePartitioningProchlorococcusPopulationsStratified1999 and @malmstromTemporalDynamicsProchlorococcus2010 found that decreased abundances of the LL clades corresponded to increased depth of the surface mixed layer. @malmstromTemporalDynamicsProchlorococcus2010 attributes the transport of LL ecotypes to the surface and consequent exposure to photoinhibitory high light levels as the reason for low cell abundances with increased mixed layer depth. @westNichePartitioningProchlorococcusPopulationsStratified1999 found the depth of the mixed layer strongly influenced the depth transition from HL to LL clades, but that factors other than light levels may influence the variations in the upper and lower depth limits of these ecotypes. We hypothesize that low cell abundances of LL ecotypes in the mixed layer is likely driven in part by increased [O~2~],  and it is [O~2~] that constrains LL clades to deeper waters, not necessarily the light level. We found that under 25 µM O~2~ representatives of 'LL' clades, SS120 and MIT9313, actually tolerate approximately 1.0 x 10^6^ µmol photons m^-2^ d^-1^ of PUR [@morelAvailableUsableStored1978],  comparable to the representative HL clade, MED4 which also exhibited growth saturation at the same cumulative diel PUR of 1.0 x 10^6^ µmol photons m^-2^ d^-1^.  Growth under lower O~2~ allowed MIT9313 to substantially increase its exploitation of higher diel PUR.

# Summary and conclusions {.unnumbered}

We analyzed  growth rates to determine the viability of *P. marinus* MED4, a clade HLI ecotype found at depths near the ocean surface,
*P. marinus* SS120, a clade LLIII ecotype found deep in the water column, and *P. marinus* MIT9313, a clade LLIV ecotype also found in deep oceans, under present day and hypothetical future niche conditions. We designed our study to account for light attenuation through ocean waters, recognizing that low light levels of the blue spectral waveband ultimately prevail in deep ocean waters [@morelAvailableUsableStored1978; @holtropVibrationalModesWater2021] and that the solar incidence angle also attenuates the photoperiod or hours of light exposure beneath the ocean surface. We also tested the growth response of *P. marinus* to varying [O~2~] under the matrix of light parameters to determine if *P. marinus* strains can cope with  diverse  ecosystem scenarios.

*Prochlorococcus marinus* MED4 has a physiological requirement for more than 4 h of light per day, and thus this strain will not exploit habitats with short photoperiods, typical of winter or light attenuated depths. MED4 is also excluded from the lowest oxygen habitats, equivalent to our 2.5 µM O~2~ experimental conditions. MED4 can, however, grow under OMZ regions with slightly higher [O~2~], as demonstrated by the growth under our 25 µM O~2~ experiments. Genomic analyses  (Figure \@ref(fig:ProchlorococcusEnzymeKms))  and previous transcriptional analyses [@bergResponsesPsbAHli2011]  suggest MED4 is excluded from growth below ~ 25 µM O~2~ because it relies upon a ubiquinol oxidase, non-electrogenic, to maintain oxidation/reduction balance in the intersystem electron transport chain.  On the other hand, MED4 shows inducible expression of FtsH isoforms [@bonisteelStrainSpecificDifferences2018], to counter photoinactivation of PSII under higher PAR and [O~2~] environments. Photoinactivation, does, however, impose an increased cost of growth upon MED4, since growth under red light, to lower the rate of photoinactivation of PSII [@murphyPhotoinactivationPhotosystemII2017], allows MED4 to achieve faster growth per absorbed photon than growth under red light. TARA Oceans
Project data [@delmontLinkingPangenomesMetagenomes2018] indeed reported presence of *P. marinus* MED4-like genomes at depths ranging from 5 m to 90 m, representing high to low blue light levels, in the Pacific South East Ocean. @delmontLinkingPangenomesMetagenomes2018 did not analyze data from depths beyond the subsurface chlorophyll maximum layer, the layer in the water column where the chlorophyll *a* concentration peaks, nor did they report the [O~2~] at those depths. Our growth findings are also consistent with Figure\@ref(fig:ProchloroPhotosynthDepthO2) showing PSII proteins annotated as MED4, at depths up to 200 meters with O~2~ of 15 µM.

*Prochlorococcus marinus* SS120, a LLIII clade representative, showed an interactive inhibition of growth by oxygen and cumulative diel PUR, with a higher tolerance for higher cumulative diel PUR under 25 µM O~2~, compared to 250 µM O~2~. Thus, SS120 can exploit habitats with O~2~ levels spanning 25 µM O~2~ to 250 µM O~2~, including high PAR environments within OMZ.  SS120 is likely excluded from higher [O~2~] and higher PAR because of genomic limitations on capacity for DNA repair (Figure \@ref(fig:DNARepairFig)), and possibly by limited capacity for synthesis of reactive oxygen quenchers (Figure  \@ref(fig:ProchlorococcusEnzymeKms)).  Our growth results are supported by @lavinNovelLineagesProchlorococcus2010 who found evidence of LLII/III
and LLIV ecotypes, using terminal restriction fragment length polymorphism analyses, at depths above 40 m, where light levels are
higher, within OMZ, and by Figure\ref(fig:ProchloroPhotosynthDepthO2) showing PSII protein subunits annotated as derived from SS120 at all depths ranging
from 20 to 200 m and all [O~2~] in an OMZ of the tropical North Pacific Ocean. SS120, grew under photoperiods longer than 4 h and showed increasing growth rate with increasing photoperiods. Therefore, SS120 can potentially exploit the longer summer photoperiods in temperate zones but again cannot sustain growth in winter through its requirement for more than 4 h of light per day, even if water temperature warms into the Clade LLIII tolerance range.

*Prochlorococcus marinus* SS120 prefers the deeper ocean where blue spectral waveband dominates at low levels, through its demonstrated ability to exploit only the lowest blue spectral waveband levels PAR of 30 µmol photons m^-2^ s^-1^ under 250 µM O~2~. However, we found *P. marinus* SS120 can exploit more diverse ecological niches within the ocean layers, even in regions with high levels of blue spectral waveband,  but only under O~2~ of 25 µM (Figure \@ref(fig:SS120PAR)). @mattisonPhotosyntheticStrategiesWild2020 and @lavinNovelLineagesProchlorococcus2010 show evidence of LLIII ecotypes at varying depths from 20 to 200 m and under all [O~2~] in the oxygen minimum zone (OMZ) of the tropical North Pacific Ocean, illustrating its tolerance of high light levels under low O~2~. SS120 has the potential to thrive in deep temperate zones, specifically during the spring, summer, and fall seasons when the duration of daylight exceeds 4 h, if O~2~ are near surface saturation of about 250 µM. Under lower oxygen levels of 25 µM, SS120 can also exploit a 4 h photoperiod in the blue waveband, and thus has the potential to inhabit a warmed, deep, temperate region during the winter season.

*Prochlorococcus marinus* MIT9313, a LLIV clade representative, shows the potential to inhabit future warmer temperate zones year-round, as it grows under a 4 h photoperiod, expected in winter or at light-attenuated depths. MIT9313 demonstrates an unexpected tolerance to higher light levels and cumulative diel PUR,  but only under low oxygen conditions of 25 µM and 2.5 µM (Figure \@ref(fig:MIT9313PAR)). The ability to utilize higher PAR and higher cumulative diel PUR under 25 µM or 2.5 µM O~2~ enables MIT9313 to grow in OMZ, even at depths closer to the surface. MIT9313 carries a gene encoding (S)-2-hydroxy-acid oxidase [@InformationEC15],  with a K~M~ for [O~2~] of ~ 250 µM (Figure \@ref(fig:ProchlorococcusEnzymeKms)), which produces H~2~O~2~.  Growth at lower [O~2~] may protect MIT9313 from auto-intoxication from production of H~2~O~2~. We hypothesize that under 250 µM O~2~ and higher blue light, *P. marinus* MIT9313 suffers photoinhibition, resulting from the inactivation of PSII caused by the production of H~2~O~2~. This photoinhibition is compounded by the limited inducible repair for PSII, due to the absence of FtsH 2 and 3  expression in *P. marinus* MIT9313 [@bonisteelStrainSpecificDifferences2018].  MIT9313 shows remarkable ability to thrive under very low [O~2~], potentially allowing it to expand into broader ecological niches. These results are supported by Figure \@ref(fig:ProchloroPhotosynthDepthO2) showing PSI protein subunits annotated as derived from MIT9313 detected at depths \> 120 m, along with PSII subunits at depths from 50 m to 200 m in regions where O~2~ was 15 µM.


# Supplemental {.unnumbered}

\newpage


```{r LabeledMC, fig.cap = LabeledMC_cap , fig.height = 8, fig.width = 6, echo = FALSE}

LabeledMC_cap <- glue("**PSI MCMIX-OD Multicultivator.**",
                      "Spectral wavebands and light levels are individually controlled for each culture tube.",
                      "Real time Optical Density (OD) measurements eliminate intrusive subsampling of sterile cultures.",
                      "The temperature of culture tubes are collectively controlled via heating or cooling of the aquarium water.",
                      "Gas with specific oxygen concentrations is bubbled through a humidifier and passed through a 0.2 um filter.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "LabeledMC.png"))
```

\newpage

```{r deltaODLogGrowthPlot, fig.cap = deltaODLogGrowthPlot_cap , fig.height = 8, fig.width = 6, echo = FALSE}

deltaODLogGrowthPlot_cap <- glue("**Fitting chlorophyll specific growth rate for each tube in the Multicultivator.**",
                      "The left y-axis is $\\Delta$OD (OD680 - OD720).",
                      "The right y-axis is actinic PAR levels (µmol photons m^-2^ s^-1^).",
                      "The x-axis is time in hours (h).",
                      "The green points are $\\Delta$OD measurements taken every 5 minutes.",
                      "The black lines are logistic growth curves fit using a nonlinear model regression (R package, minpack.lm). ",
                      "The gold points are the residuals of the fit.",
                      "The blue or orange points represent the actinic spectral waveband under 450 nm or 660 nm, showing the imposed PAR level (µmol photons m^-2^ s^-1^) and photoperiod (h).",
                      "Meta data associated with each Multicultivator tube are in columns.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "deltaODLogGrowthPlot.png"))
```

\newpage

```{r MED4OverlayPlots, fig.cap = MED4OverlayPlots_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MED4OverlayPlots_cap <- glue("**Normalized absorbance and Photosynthetically Usable Radiation for *Prochlorococcus marinus* MED4 grown under three emission wavebands.**",
                       "**A.** Growth light emission spectra from the White LED (normalized to 439 nm; dotted black line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line); and calculated PUR spectra (solid black line and shaded grey).",
                       "**B.** Growth light emission spectra at 660 nm (normalized to 647 nm; dotted red line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line);  and calculated PUR spectra (solid black line and shaded red).", 
                       "**C.**  Growth light emission spectra at 450 nm (normalized to 441 nm; dotted blue line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line);  and calculated PUR spectra (solid black line and shaded blue).",  
                       
                       "Actinic PAR (µmol photons m^-2^ s^-1^) and calculated PUR (µmol photons m^-2^ s^-1^) levels are indicated.",

                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MED4OverlayPlots.png"))


```

\newpage

```{r SS120OverlayPlots, fig.cap = SS120OverlayPlots_cap, fig.height = 6, fig.width = 8, echo = FALSE}

SS120OverlayPlots_cap <- glue("**Normalized absorbance and Photosynthetically Usable Radiation for *Prochlorococcus marinus* SS120 grown under three emission wavebands.",
                       "**A.** Growth light emission spectra from the White LED (normalized to 439 nm; dotted black line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line); and calculated PUR spectra (solid black line and shaded grey).",
                       "**B.** Growth light emission spectra at 660 nm (normalized to 647 nm; dotted red line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line);  and calculated PUR spectra (solid black line and shaded red).",  
                       "**C.**  Growth light emission spectra at 450 nm (normalized to 441 nm; dotted blue line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line);  and calculated PUR spectra (solid black line and shaded blue).",  
                       "Actinic PAR (µmol photons m^-2^ s^-1^) and calculated PUR (µmol photons m^-2^ s^-1^) levels are indicated.",
                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "SS120OverlayPlots.png"))


```


\newpage

```{r MIT9313OverlayPlots, fig.cap = MIT9313OverlayPlots_cap, fig.height = 6, fig.width = 8, echo = FALSE}

MIT9313OverlayPlots_cap <- glue("**Normalized absorbance and Photosynthetically Usable Radiation for *Prochlorococcus marinus* MIT9313 grown under three emission wavebands.**",
                                "**A.** Growth light emission spectra from the White LED (normalized to 439 nm; dotted black line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line); and calculated PUR spectra (solid black line and shaded grey).",
                       "**B.** Growth light emission spectra at 660 nm (normalized to 647 nm; dotted red line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line);  and calculated PUR spectra (solid black line and shaded red).",  
                       "**C.**  Growth light emission spectra at 450 nm (normalized to 441 nm; dotted blue line); whole cell absorbance spectra (normalized to absorbance maxima between 400 nm and 460 nm; dashed purple line);  and calculated PUR spectra (solid black line and shaded blue).",  
                       "Actinic PAR (µmol photons m^-2^ s^-1^) and calculated PUR (µmol photons m^-2^ s^-1^) levels are indicated.",
                       
                       .sep = " ")

knitr::include_graphics(file.path(FiguresIn, "MIT9313OverlayPlots.png"))


```


```{r Generate GrowthTable, include = FALSE }

MaxGrowthTable <- PURPARFits %>%
  filter(Par_ue %in% c(30,90,180),
         Strain %in% c("MED4", "SS120" ,"MIT9313")) %>%
  select(Strain, O2_uM, ReplicateMean_deltaOD_Lmu_corr_day, Photoperiod, Par_ue, WL ) %>%
  group_by(Strain, O2_uM) %>%
  summarize(MaxGrowth = round(max(ReplicateMean_deltaOD_Lmu_corr_day), 2), 
            Photoperiod = Photoperiod[which.max(ReplicateMean_deltaOD_Lmu_corr_day)], 
            Par_ue = Par_ue[which.max(ReplicateMean_deltaOD_Lmu_corr_day)], 
            WL= WL[which.max(ReplicateMean_deltaOD_Lmu_corr_day)]) %>%
  ungroup %>% 
 mutate(Strain = factor(Strain, levels = c("MED4", "SS120", "MIT9313")),
         O2_uM = factor(O2_uM, levels = c(250, 25, 2.5)),
         WL = case_when(WL %in% c(450, 530, 660) ~ WL,
                              WL %in% c("WW") ~ paste("White LED", sep = " "))) %>% 
  mutate(Photoperiod = ifelse(MaxGrowth != 0, Photoperiod,  paste("All tested", sep = " ")),
         Par_ue = ifelse(MaxGrowth != 0, Par_ue, paste("All tested", sep = " ")),
          WL = ifelse(MaxGrowth != 0, WL, paste("All tested", sep = " "))) %>% 
   relocate(WL, .after = Par_ue) %>%
   relocate(MaxGrowth, .after = WL) %>% 
   relocate(O2_uM, .after = WL) %>% 
  arrange(Strain, O2_uM)

```

```{r MaxGrowthTable, fig.height = 6, fig.width = 8, echo = FALSE }


knitr::kable(MaxGrowthTable,
             caption = "The maximum growth rate, µ~max~ (d^-1^) in each O~2~ experiment for each strain. The corresponding photoperiod, PAR level and spectral waveband are indicated.",
            col.names =  c("Strain",
                          "Photoperiod (h)",
                          "PAR (µmol photons m^-2^ s^-1^)",
                          "Spectral waveband (nm)",
                          "[O~2~] (µM)",
                          "µ~max~ (d^-1^)"),
              align = "lccccc"
             )
```

\newpage

```{r AllStrainPurFitsPlots,  fig.cap = AllStrainPurFitsPlots_cap, fig.height = 10, fig.width = 10, echo = FALSE}

AllStrainPurFitsPlots_cap <- glue("**Chlorophyll specific growth rate (d^-1^) vs. cumulative diel Photosynthetic Usable Radiation (PUR, µmol photons m^-2^ d^-1^).**",
                    "Rows show levels of imposed dissolved O~2~ concentrations as 250 µM, 25 µM and 2.5 µM.",
                    "Columns are strains; MED4, SS120 and MIT9313.",
                    "Shapes show the imposed photoperiod (h); 4 h (solid square),  8 h (solid diamond), 12 h (solid circle), 16 h (solid upright triangle).",
                      "Symbol colours show the spectral waveband for growth; white LED (black symbols), 660 nm (red symbols), and 450 nm (blue symbols).",
                      "Large symbols show mean of growth rate from logistic curve fits; small symbols show values for replicate determinations, if any.",
                    "Platt (add Platt citation here) 3 parameter model fit to data pooled for each combination of strain and dissolved oxygen shown with solid lines. Separate models fit to photoperiod data and shown if significantly different from the pooled model; 4 h (long dashed line); 8 h (dotted line); 12 h (dashed line); and 16 h (dot dashed line).",
                       .sep = " ")


knitr::include_graphics(file.path(FiguresIn, "AllStrainPurFitsPlots.png"))

```

\newpage

# Acknowledgements  {.unnumbered}
Jonah Sheinin (Mount Allison Student) assisted with growth of some cultures for protein analyses, while Carlie Barnhill (Mount Allison Student) assisted with code for import of multicultivator growth data files.


# Funding  {.unnumbered}
To Be Entered through PLoSONe system and deleted here
Czech Academy of Science visiting fellowship supporting DAC work at AlgaTech (OP)
Canada Research Chair in Phytoplankton Ecophysiology (DAC)
Natural Sciences and Engineering Research Council of Canada, 'Latitude and Light' (DAC)
Canada Foundation for Innovation (DAC)
New Brunswick Foundation for Innovation (DAC)
Rice Graduate Fellowship (MS)

# References {.unnumbered}